Define data and savegame directory.
Index: spacezero-0.86.00/include/general.h
===================================================================
--- spacezero-0.86.00.orig/include/general.h	2013-12-22 21:46:56.453389163 +0100
+++ spacezero-0.86.00/include/general.h	2013-12-22 21:46:56.449389164 +0100
@@ -111,7 +111,7 @@
 #define MAXASTEROIDDISTANCE2 784000000 /* max distance2 an asteroid can be seen (20 sectors )*/
 
 /* FILES */
-#define SAVEDIR ".spacezero"
+#define SAVEDIR ".config/spacezero"
 #define SAVETMPFILE "/tmp/tmpspacezero"
 #define SAVEFILE0 "save"
 #define SAVEFILE1 "savecl"
