Just happens :)
Index: spacezero-0.83.28/src/data.c
===================================================================
--- spacezero-0.83.28.orig/src/data.c	2012-10-04 16:02:53.000000000 +0200
+++ spacezero-0.83.28/src/data.c	2012-11-08 19:38:13.459997088 +0100
@@ -429,7 +429,7 @@
   n=strlen(cad);
   if(n>32)n=32; /* max of log messages */
 
-  if(mode==1){ /* check if is recently added => dont add*/
+  if(mode==1){ /* check if is recently added => don't add*/
     m=0;
     list=hlist->next;
     while(list!=NULL){
Index: spacezero-0.83.28/src/spacezero.c
===================================================================
--- spacezero-0.83.28.orig/src/spacezero.c	2012-11-06 18:28:04.000000000 +0100
+++ spacezero-0.83.28/src/spacezero.c	2012-11-08 19:38:13.459997088 +0100
@@ -1823,7 +1823,7 @@
   navcontrol=TRUE;
   if(key->order.state==TRUE)navcontrol=FALSE;
   
-  /* in map view dont control the ship*/ 
+  /* in map view don't control the ship*/ 
   if(gdraw.map==TRUE){
     key->accel.state=key->turnleft.state=key->turnright.state=key->fire.state=FALSE;
     key->automode.state=key->manualmode.state=FALSE;
