### Search in the correct game directory.
Index: srb2-2.0.6/src/sdl/i_system.c
===================================================================
--- srb2-2.0.6.orig/src/sdl/i_system.c	2011-09-01 20:19:26.000000000 +0200
+++ srb2-2.0.6/src/sdl/i_system.c	2013-05-12 02:21:08.417087331 +0200
@@ -180,7 +180,7 @@
 #elif defined (__unix__) || defined(__APPLE__) || defined (UNIXCOMMON)
 #define DEFAULTWADLOCATION1 "/usr/local/share/games/SRB2"
 #define DEFAULTWADLOCATION2 "/usr/local/games/SRB2"
-#define DEFAULTWADLOCATION3 "/usr/share/games/SRB2"
+#define DEFAULTWADLOCATION3 "/usr/share/games/srb2/SRB2"
 #define DEFAULTWADLOCATION4 "/usr/games/SRB2"
 #define DEFAULTSEARCHPATH1 "/usr/local/games"
 #define DEFAULTSEARCHPATH2 "/usr/games"
