Just happens :)
Index: warsow-1.5/source/source/client/cl_serverlist.c
===================================================================
--- warsow-1.5.orig/source/source/client/cl_serverlist.c	2014-05-07 20:59:02.216291061 +0200
+++ warsow-1.5/source/source/client/cl_serverlist.c	2014-05-07 20:59:02.216291061 +0200
@@ -296,7 +296,7 @@
 	requeststring = va( cmdname );
 
 	// send a broadcast packet
-	Com_DPrintf( "quering %s...\n", server );
+	Com_DPrintf( "querying %s...\n", server );
 
 	if( NET_StringToAddress( server, &adr ) )
 	{
Index: warsow-1.5/source/source/client/cl_sound.c
===================================================================
--- warsow-1.5.orig/source/source/client/cl_sound.c	2014-05-07 20:59:02.216291061 +0200
+++ warsow-1.5/source/source/client/cl_sound.c	2014-05-07 20:59:02.216291061 +0200
@@ -169,7 +169,7 @@
 	}
 
 	if( verbose )
-		Com_Printf( "Initialization of %s succesful\n", name );
+		Com_Printf( "Initialization of %s successful\n", name );
 
 	return qtrue;
 }
