Just happens :)
Index: warsow-2.0/source/source/client/cl_serverlist.c
===================================================================
--- warsow-2.0.orig/source/source/client/cl_serverlist.c	2015-12-12 01:38:37.936594067 +0100
+++ warsow-2.0/source/source/client/cl_serverlist.c	2015-12-12 01:38:37.932593982 +0100
@@ -296,7 +296,7 @@
 	}
 
 	// send a broadcast packet
-	Com_DPrintf( "quering %s...\n", server );
+	Com_DPrintf( "querying %s...\n", server );
 
 	if( NET_StringToAddress( server, &adr ) )
 	{
Index: warsow-2.0/source/source/client/cl_sound.c
===================================================================
--- warsow-2.0.orig/source/source/client/cl_sound.c	2015-12-12 01:38:37.936594067 +0100
+++ warsow-2.0/source/source/client/cl_sound.c	2015-12-12 01:38:37.932593982 +0100
@@ -172,7 +172,7 @@
 	}
 
 	if( verbose )
-		Com_Printf( "Initialization of %s succesful\n", name );
+		Com_Printf( "Initialization of %s successful\n", name );
 
 	return true;
 }
