# Just typos.
Index: alephone-20130707/Source_Files/FFmpeg/SDL_ffmpeg.c
===================================================================
--- alephone-20130707.orig/Source_Files/FFmpeg/SDL_ffmpeg.c	2013-07-05 02:33:29.000000000 +0200
+++ alephone-20130707/Source_Files/FFmpeg/SDL_ffmpeg.c	2013-07-08 18:56:17.296034580 +0200
@@ -1309,7 +1309,7 @@
     }
     else
     {
-        SDL_ffmpegSetError( "could not retreive frame rate from stream" );
+        SDL_ffmpegSetError( "could not retrieve frame rate from stream" );
 
         if ( nominator ) *nominator = 0;
 
