# fix path on non windows systems
Index: stratagus-2.4.0/gameheaders/stratagus-game-launcher.h
===================================================================
--- stratagus-2.4.0.orig/gameheaders/stratagus-game-launcher.h	2016-01-15 16:13:49.000000000 +0100
+++ stratagus-2.4.0/gameheaders/stratagus-game-launcher.h	2016-02-02 21:26:25.887038839 +0100
@@ -103,14 +103,6 @@
  * Path to stratagus executable binary
  **/
 
-/**
- * \def TITLE_PNG
- * OPTIONAL: Path to title screen (for testing if data was extracted)
- **/
-#ifndef TITLE_PNG
-#define TITLE_PNG "%s\\graphics\\ui\\title.png"
-#endif
-
 /* Fake definitions for Doxygen */
 #ifdef DOXYGEN
 #define GAME_NAME
@@ -129,6 +121,18 @@
 #define WIN32
 #endif
 
+/**
+ * \def TITLE_PNG
+ * OPTIONAL: Path to title screen (for testing if data was extracted)
+ **/
+#ifndef TITLE_PNG
+#ifdef WIN32
+#define TITLE_PNG "%s\\graphics\\ui\\title.png"
+#else
+#define TITLE_PNG "%s/graphics/ui/title.png"
+#endif
+#endif
+
 #ifndef WIN32
 #if ! defined (DATA_PATH) || ! defined (SCRIPTS_PATH) || ! defined (STRATAGUS_BIN)
 #error You need to define paths, see stratagus-game-launcher.h
