# install everything in /usr/games
Index: stratagus-2.4.1/CMakeLists.txt
===================================================================
--- stratagus-2.4.1.orig/CMakeLists.txt	2016-11-07 19:08:52.565290514 +0100
+++ stratagus-2.4.1/CMakeLists.txt	2016-11-07 19:08:52.561290411 +0100
@@ -1237,10 +1237,10 @@
 ########### install files ###############
 
 install(TARGETS stratagus DESTINATION ${GAMEDIR})
-install(TARGETS png2stratagus DESTINATION ${BINDIR})
+install(TARGETS png2stratagus DESTINATION ${GAMEDIR})
 
 if(SQLITE_FOUND)
-	install(TARGETS metaserver DESTINATION ${SBINDIR})
+	install(TARGETS metaserver DESTINATION ${GAMEDIR})
 endif()
 
 if(ENABLE_DOC AND DOXYGEN_FOUND)
