Index: soulfu-1.5/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ soulfu-1.5/Makefile	2013-10-14 22:44:22.856488635 +0200
@@ -0,0 +1,22 @@
+CC = gcc
+BIN = soulfu
+INCLUDES = -Isourcecode `sdl-config --cflags` -I/usr/include/GL
+ifeq ($(DEB_BUILD_ARCH),amd64)
+FLAGS = -O0 -g -m32
+LIBS = -L/usr/lib32 -lSDL-1.2 -lSDL_net-1.2 -lGL -ljpeg -lvorbis -lm
+else
+FLAGS = -O0 -g
+LIBS = `sdl-config --libs` -lSDL_net -lGL -ljpeg -lvorbis -lm -logg
+endif
+
+all: $(BIN)
+
+$(BIN):
+ifeq ($(DEB_BUILD_ARCH),amd64)
+	touch $@
+else
+	$(CC) $(FLAGS) $(INCLUDES) sourcecode/soulfu.c -o $@ $(LIBS)
+endif
+
+clean:
+	rm -f $(BIN)
