Setting platform to linux.
Index: bitfighter-0.0.018/lua/lua-vec/Makefile
===================================================================
--- bitfighter-0.0.018.orig/lua/lua-vec/Makefile	2013-02-18 18:33:58.000000000 +0100
+++ bitfighter-0.0.018/lua/lua-vec/Makefile	2013-02-18 18:36:09.890677256 +0100
@@ -5,7 +5,7 @@
 # == CHANGE THE SETTINGS BELOW TO SUIT YOUR ENVIRONMENT =======================
 
 # Your platform. See PLATS for possible values.
-PLAT= none
+PLAT=linux
 
 # Where to install. The installation starts in the src and doc directories,
 # so take care if INSTALL_TOP is not an absolute path.
Index: bitfighter-0.0.018/lua/lua-vec/src/Makefile
===================================================================
--- bitfighter-0.0.018.orig/lua/lua-vec/src/Makefile	2013-02-18 18:33:58.000000000 +0100
+++ bitfighter-0.0.018/lua/lua-vec/src/Makefile	2013-02-18 18:36:50.146676058 +0100
@@ -9,7 +9,7 @@
 ifeq ($(findstring Windows, $(OS)), Windows)
 PLAT= mingw
 else
-PLAT= linux
+PLAT=linux
 endif
 
 CC= gcc
