Description: Enable experimental support for link-time optimization
Author: Sven Eckelmann <sven@narfation.org>

---
diff --git a/projects/unix/Makefile b/projects/unix/Makefile
index b50545b17f44365586c9d21d5500f6728e34f32b..5deda7ea62e3e86aac9a2ad3681249a46e19ad4e 100644
--- a/projects/unix/Makefile
+++ b/projects/unix/Makefile
@@ -249,6 +249,7 @@ targets:
 	@echo "    DESTDIR=path  == path to prepend to all installation paths (only for packagers)"
 	@echo "  Debugging Options:"
 	@echo "    DEBUG=1       == add debugging symbols"
+	@echo "    LTO=1         == enable experimental build with link-time optimization"
 	@echo "    V=1           == show verbose compiler output"
 
 all: $(TARGET)
@@ -270,6 +271,11 @@ CFLAGS += -MD
 -include $(OBJECTS:.o=.d)
 
 CXXFLAGS += $(CFLAGS)
+ifeq ($(LTO), 1)
+  CFLAGS += -flto
+  CXXFLAGS += -flto
+  LDFLAGS += -fuse-linker-plugin $(CXXFLAGS)
+endif
 
 # standard build rules
 $(OBJDIR)/%.o: $(SRCDIR)/%.c
