Description: Readded jpeg task 278 used during Zelda OOT boot
Author: Bobby Smiles <bobby.smiles32@gmail.com>
Origin: upstream, https://bitbucket.org/richard42/mupen64plus-rsp-hle/changeset/27234e46cfe5

---
diff --git a/src/jpeg.c b/src/jpeg.c
index 67f5ef489d6a9f8009484eb29a0402afc792b0b8..c51e611200322cebf03d6c6ade8e0c94613bffd0 100644
--- a/src/jpeg.c
+++ b/src/jpeg.c
@@ -364,7 +364,7 @@ void ps_jpg_uncompress(OSTask_t *task)
         }
 
         // Texel Formatting (RGBA16)
-        offset = ps_jpg_data.pMacroBlocks + oMBsize*mb;
+        offset = ps_jpg_data.pMacroBlocks + iMBsize*mb;
         y_offset = 0;
         u_offset = oMBsize/2;
 
diff --git a/src/main.c b/src/main.c
index 0142d781bdfd2069b71e3a786fc1a242c915c077..c72c3422233a9b6ce0785e0b35ac87b9e091eda8 100644
--- a/src/main.c
+++ b/src/main.c
@@ -302,7 +302,10 @@ EXPORT unsigned int CALL DoRspCycles(unsigned int Cycles)
             {
                 switch(sum)
                 {
-                case 0x2caa6: // Pokemon Stadium {1,2} jpg decompression
+                case 0x278: // Zelda OOT during boot
+                  taskdone();
+                  return Cycles;
+                case 0x2caa6: // Zelda OOT, Pokemon Stadium {1,2} jpg decompression
                     ps_jpg_uncompress(task);
                     taskdone();
                     return Cycles;
