From 93292d80924aeadfda9f327ba78891cc4bab6c3c Mon Sep 17 00:00:00 2001
From: koda <vittorio.giovara@gmail.com>
Date: Mon, 16 Nov 2015 13:59:04 +0100
Subject: [PATCH] avwrapper: Adjust prefixed symobols version guard

---
 hedgewars/avwrapper/avwrapper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hedgewars/avwrapper/avwrapper.c b/hedgewars/avwrapper/avwrapper.c
index 9ed00b4..3441f0f 100644
--- a/hedgewars/avwrapper/avwrapper.c
+++ b/hedgewars/avwrapper/avwrapper.c
@@ -79,7 +79,7 @@ static void rescale_ts(AVPacket *pkt, AVRational ctb, AVRational stb)
 }
 #endif
 
-#if LIBAVCODEC_VERSION_MAJOR < 57
+#if LIBAVCODEC_VERSION_MAJOR < 56
 #define AV_CODEC_CAP_DELAY                  CODEC_CAP_DELAY
 #define AV_CODEC_CAP_VARIABLE_FRAME_SIZE    CODEC_CAP_VARIABLE_FRAME_SIZE
 #define AV_CODEC_FLAG_GLOBAL_HEADER         CODEC_FLAG_GLOBAL_HEADER
