From f121e6c350a041a429a57ba4748ad673aa8420ea Mon Sep 17 00:00:00 2001
From: koda <vittorio.giovara@gmail.com>
Date: Mon, 16 Nov 2015 18:28:10 +0100
Subject: [PATCH] avwrapper: Explicitly check for macro existence

---
 hedgewars/avwrapper/avwrapper.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/hedgewars/avwrapper/avwrapper.c b/hedgewars/avwrapper/avwrapper.c
index 3441f0f..346b3a8 100644
--- a/hedgewars/avwrapper/avwrapper.c
+++ b/hedgewars/avwrapper/avwrapper.c
@@ -79,10 +79,16 @@ static void rescale_ts(AVPacket *pkt, AVRational ctb, AVRational stb)
 }
 #endif
 
-#if LIBAVCODEC_VERSION_MAJOR < 56
+#ifndef AV_CODEC_CAP_DELAY
 #define AV_CODEC_CAP_DELAY                  CODEC_CAP_DELAY
+#endif
+#ifndef AV_CODEC_CAP_VARIABLE_FRAME_SIZE
 #define AV_CODEC_CAP_VARIABLE_FRAME_SIZE    CODEC_CAP_VARIABLE_FRAME_SIZE
+#endif
+#ifndef AV_CODEC_FLAG_GLOBAL_HEADER
 #define AV_CODEC_FLAG_GLOBAL_HEADER         CODEC_FLAG_GLOBAL_HEADER
+#endif
+#ifndef AV_CODEC_FLAG_QSCALE
 #define AV_CODEC_FLAG_QSCALE                CODEC_FLAG_QSCALE
 #endif
 
