Name the linking library properly.
Index: goollie-1.3/src/CMakeLists.txt
===================================================================
--- goollie-1.3.orig/src/CMakeLists.txt	2009-06-14 22:15:14.737446586 +0200
+++ goollie-1.3/src/CMakeLists.txt	2009-06-14 22:25:29.744455472 +0200
@@ -12,7 +12,7 @@
 -lpthread
 -lz
 -lutil
-libtuxcap.so.4.0)
+-ltuxcap)
 
 SET_TARGET_PROPERTIES(${CurrentExe}
                              PROPERTIES
