--- a/src/install/nix/redeclipse.desktop
+++ b/src/install/nix/redeclipse.desktop
@@ -21,7 +21,5 @@
 Comment[pt]=Jogo de tiros em primeira pessoa com jogabilidade ágil e editor incorporado, modificada para Debian.
 Comment[sv]=Förstapersonsskjutare med rörlig spelstil och inbyggd baneditor, anpassad för Debian.
 Icon=redeclipse
-TryExec=redeclipse
-Exec=bin/reclient -r
-Path=/usr/share/redeclipse
+Exec=/usr/games/redeclipse
 Categories=Game;ActionGame;
