Description: Update paths for Debian
Author: Scott Howard <showard314@gmail.com>

Index: sandboxgamemaker-2.8.1/src/launcher/pasl.cpp
===================================================================
--- sandboxgamemaker-2.8.1.orig/src/launcher/pasl.cpp	2012-11-04 08:12:52.000000000 +0100
+++ sandboxgamemaker-2.8.1/src/launcher/pasl.cpp	2012-11-18 15:02:09.690663717 +0100
@@ -461,28 +461,28 @@
         if(wxIsPlatform64Bit())
         {
             //LinExe = wxT("./bin//sandbox_client_x86_64 -r ");
-            CurFPSc = wxT("./bin/sandbox_client_64_fps -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurSSPc = wxT("./bin/sandbox_client_64_ssp -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurRPGc = wxT("./bin/sandbox_client_64_rpg -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurFPSs = wxT("./bin/sandbox_server_64_fps ");
-            CurSSPs = wxT("./bin/sandbox_server_64_ssp ");
-            CurRPGs = wxT("./bin/sandbox_server_64_rpg ");
-            CurMovie = wxT("./bin/sandbox_client_32_movie");
+            CurFPSc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_fps -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurSSPc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_ssp -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurRPGc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_rpg -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurFPSs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_fps ");
+            CurSSPs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_ssp ");
+            CurRPGs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_rpg ");
+            CurMovie = wxT("/usr/lib/sandboxgamemaker/sandbox_client_movie");
         }
         else
         {
-            CurFPSc = wxT("./bin/sandbox_client_32_fps -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurSSPc = wxT("./bin/sandbox_client_32_ssp -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurRPGc = wxT("./bin/sandbox_client_32_rpg -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
-            CurFPSs = wxT("./bin/sandbox_server_32_fps ");
-            CurSSPs = wxT("./bin/sandbox_server_32_ssp ");
-            CurRPGs = wxT("./bin/sandbox_server_32_rpg ");
-            CurMovie = wxT("./bin/sandbox_client_32_movie");
+            CurFPSc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_fps -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurSSPc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_ssp -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurRPGc = wxT("/usr/lib/sandboxgamemaker/sandbox_client_rpg -q") + wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts -r ");
+            CurFPSs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_fps ");
+            CurSSPs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_ssp ");
+            CurRPGs = wxT("/usr/lib/sandboxgamemaker/sandbox_server_rpg ");
+            CurMovie = wxT("/usr/lib/sandboxgamemaker/sandbox_client_movie");
             //LinExe = wxT(".//bin//sandbox_client_i686 -r ");
         }
         //LinExe = wxT(".//bin//sandbox_client_") + wxGetCommandOutput(wxT("uname -m")) + " -r ");
         //CurExe = LinExe;
-        MapDir = (wxGetCwd() + wxT("/packages/base/"));
+        MapDir = (wxT("/usr/share/sandboxgamemaker/packages/base/"));
 	StuffDir = (wxStandardPaths::Get().GetUserConfigDir() + wxT("/.platinumarts/packages/base/"));
     #endif
 
Index: sandboxgamemaker-2.8.1/src/Makefile
===================================================================
--- sandboxgamemaker-2.8.1.orig/src/Makefile	2012-11-04 08:12:52.000000000 +0100
+++ sandboxgamemaker-2.8.1/src/Makefile	2012-11-18 15:02:36.002662935 +0100
@@ -219,9 +219,8 @@
 
 pasl:   $(LAUNCHER_OBJS)
 	$(CXX) $(CXXFLAGS) -o launch $(LAUNCHER_OBJS) $(LAUNCHER_LIBS)
-	cp launch ../launcher
-	strip ../launcher
-	chmod +x ../launcher
+	cp launch ../sandboxgamemaker_glauncher
+	chmod +x ../sandboxgamemaker_glauncher
 
 all: client server
 
