# executables have a _linux suffix in the bin dir.
Index: redeclipse-1.5.0/src/system-install.mk
===================================================================
--- redeclipse-1.5.0.orig/src/system-install.mk	2015-03-28 00:05:37.438558273 +0100
+++ redeclipse-1.5.0/src/system-install.mk	2015-03-28 00:49:57.770479135 +0100
@@ -2,8 +2,8 @@
 appnamefull=$(shell sed -n 's/.define VERSION_NAME *"\([^"]*\)"/\1/p' version.h)
 appsrcname=$(APPNAME)
 cappname=$(shell echo $(appname) | tr '[:lower:]' '[:upper:]')# Captial appname
-appclient=$(APPCLIENT)
-appserver=$(APPSERVER)
+appclient=$(APPCLIENT)_linux
+appserver=$(APPSERVER)_linux
 
 prefix=/usr/local
 games=
