Description: Dumb down unneeded wrappers for compatibillity on macs
Author: Scott Howard <showard@debian.org>

Index: triplea-1.8.0.5/src/games/strategy/common/ui/BasicGameMenuBar.java
===================================================================
--- triplea-1.8.0.5.orig/src/games/strategy/common/ui/BasicGameMenuBar.java	2015-01-02 21:51:52.967831455 +0100
+++ triplea-1.8.0.5/src/games/strategy/common/ui/BasicGameMenuBar.java	2015-01-02 21:51:52.963831455 +0100
@@ -550,7 +550,7 @@
 				}
 			}).setMnemonic(KeyEvent.VK_A);
 		}
-		else
+/*		else
 		// On Mac OS X, put the About menu where Mac users expect it to be
 		{
 			Application.getApplication().addApplicationListener(new ApplicationAdapter()
@@ -563,6 +563,7 @@
 				}
 			});
 		}
+*/
 	}
 	
 	/**
Index: triplea-1.8.0.5/src/games/strategy/common/ui/MacWrapper.java
===================================================================
--- triplea-1.8.0.5.orig/src/games/strategy/common/ui/MacWrapper.java	2015-01-02 21:51:52.967831455 +0100
+++ triplea-1.8.0.5/src/games/strategy/common/ui/MacWrapper.java	2015-01-02 21:53:36.199828384 +0100
@@ -32,7 +32,7 @@
 public class MacWrapper
 {
 	private static MainGameFrame s_shutdownFrame;
-	static
+	/*static
 	{
 		Application.getApplication().addApplicationListener(new ApplicationAdapter()
 		{
@@ -46,7 +46,7 @@
 			}
 		});
 	}
-	
+*/
 	// keep this in its own class, otherwise we get a no class def error when
 	// we try to load the game and the stubs arent in the classpath
 	// i think the java validator triggers this
Index: triplea-1.8.0.5/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java
===================================================================
--- triplea-1.8.0.5.orig/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java	2015-01-02 21:51:52.967831455 +0100
+++ triplea-1.8.0.5/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java	2015-01-02 21:54:06.011827497 +0100
@@ -26,19 +26,5 @@
 	// keep this in its own class, otherwise we get a no class def error when
 	// we try to load the game and the stubs arent in the classpath
 	// i think the java validator triggers this
-	public static void registerMacShutdownHandler(final LobbyFrame frame)
-	{
-		final Application application = Application.getApplication();// new Application();
-		application.addApplicationListener(new ApplicationAdapter()
-		{
-			@Override
-			public void handleQuit(final ApplicationEvent event)
-			{
-				if (frame != null)
-					frame.shutdown();
-				else
-					System.exit(0);
-			}
-		});
-	}
+	public static void registerMacShutdownHandler(final LobbyFrame frame){}
 }
