Description: Dumb down unneeded wrappers for compatibillity on macs
Author: Scott Howard <showard@debian.org>

Index: triplea-1.8.0.7/src/games/strategy/common/ui/BasicGameMenuBar.java
===================================================================
--- triplea-1.8.0.7.orig/src/games/strategy/common/ui/BasicGameMenuBar.java	2015-08-16 19:56:27.847768515 +0200
+++ triplea-1.8.0.7/src/games/strategy/common/ui/BasicGameMenuBar.java	2015-08-16 19:57:01.964464927 +0200
@@ -435,7 +435,7 @@
               JOptionPane.PLAIN_MESSAGE);
         }
       }).setMnemonic(KeyEvent.VK_A);
-    } else
+    } /*else
     // On Mac OS X, put the About menu where Mac users expect it to be
     {
       Application.getApplication().addApplicationListener(new ApplicationAdapter() {
@@ -446,7 +446,7 @@
               JOptionPane.PLAIN_MESSAGE);
         }
       });
-    }
+    }*/
   }
 
 
Index: triplea-1.8.0.7/src/games/strategy/common/ui/MacWrapper.java
===================================================================
--- triplea-1.8.0.7.orig/src/games/strategy/common/ui/MacWrapper.java	2015-08-16 19:56:27.847768515 +0200
+++ triplea-1.8.0.7/src/games/strategy/common/ui/MacWrapper.java	2015-08-16 19:57:51.769479780 +0200
@@ -14,7 +14,7 @@
 public class MacWrapper {
   private static MainGameFrame s_shutdownFrame;
 
-  static {
+  /*static {
     Application.getApplication().addApplicationListener(new ApplicationAdapter() {
       @Override
       public void handleQuit(final ApplicationEvent event) {
@@ -25,7 +25,7 @@
         }
       }
     });
-  }
+  }*/
 
   // keep this in its own class, otherwise we get a no class def error when
   // we try to load the game and the stubs arent in the classpath
Index: triplea-1.8.0.7/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java
===================================================================
--- triplea-1.8.0.7.orig/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java	2015-08-16 19:56:27.847768515 +0200
+++ triplea-1.8.0.7/src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java	2015-08-16 19:59:34.175560520 +0200
@@ -11,7 +11,7 @@
   // keep this in its own class, otherwise we get a no class def error when
   // we try to load the game and the stubs arent in the classpath
   // i think the java validator triggers this
-  public static void registerMacShutdownHandler(final LobbyFrame frame) {
+  /*public static void registerMacShutdownHandler(final LobbyFrame frame) {
     final Application application = Application.getApplication();// new Application();
     application.addApplicationListener(new ApplicationAdapter() {
       @Override
@@ -23,5 +23,6 @@
         }
       }
     });
-  }
+  }*/
+  public static void registerMacShutdownHandler(final LobbyFrame frame){}
 }
