Index: jngl-1.2.0/src/freetype.hpp
===================================================================
--- jngl-1.2.0.orig/src/freetype.hpp	2016-11-06 16:30:36.479507169 +0100
+++ jngl-1.2.0/src/freetype.hpp	2016-11-06 16:31:52.697050194 +0100
@@ -9,7 +9,10 @@
 #include "texture.hpp"
 
 #include <ft2build.h>
-#include FT_FREETYPE_H
+#include <freetype2/freetype/freetype.h>
+#include <freetype2/freetype/ftglyph.h>
+#include <freetype2/freetype/ftoutln.h>
+#include <freetype2/freetype/fttrigon.h>
 
 #include <vector>
 #include <string>
