# build against png16
Index: jngl-1.2.0/SConstruct
===================================================================
--- jngl-1.2.0.orig/SConstruct	2016-11-06 16:35:34.441753970 +0100
+++ jngl-1.2.0/SConstruct	2016-11-06 16:35:34.437753883 +0100
@@ -115,7 +115,7 @@
 	env.Append(LIBPATH='.', CPPPATH='src')
 	testEnv = env.Clone()
 	testEnv.ParseConfig("pkg-config --cflags --libs jngl.pc")
-	testEnv.Program('test', testSrc, CPPFLAGS="-std=c++0x", LIBS=Split("jngl GL GLU jpeg openal webp png12 glib-2.0 Xxf86vm freetype fontconfig vorbisfile GLEW vorbis m ogg X11"))
+	testEnv.Program('test', testSrc, CPPFLAGS="-std=c++0x", LIBS=Split("jngl GL GLU jpeg openal webp png16 glib-2.0 Xxf86vm freetype fontconfig vorbisfile GLEW vorbis m ogg X11"))
 	if env['python']:
 		env = env.Clone()
 		env.ParseConfig("pkg-config --cflags --libs jngl.pc")
