### raring has python3.3. Change accordingly.
Index: jngl-1.1.0/SConstruct
===================================================================
--- jngl-1.1.0.orig/SConstruct	2013-05-08 20:49:25.000000000 +0200
+++ jngl-1.1.0/SConstruct	2013-05-08 21:06:32.840405664 +0200
@@ -103,9 +103,9 @@
 	if env['python']:
 		env = env.Clone()
 		env.ParseConfig("pkg-config --cflags --libs jngl.pc")
-		env.Append(CPPPATH="/usr/include/python3.2",
+		env.Append(CPPPATH="/usr/include/python3.3m",
 		           LIBPATH=Split("src ./python"),
-		           LIBS=Split("python3.2mu boost_python-py32"),
+		           LIBS=Split("python3.3m boost_python-py33"),
 		           CPPFLAGS="-std=c++11")
 		env.SharedLibrary(target="python/libjngl.so",
 		                  source="python/main.cpp")
