# Don't use installers which are not for Linux.
Index: jngl-1.1.0/SConstruct
===================================================================
--- jngl-1.1.0.orig/SConstruct	2013-02-16 13:03:27.284284604 +0100
+++ jngl-1.1.0/SConstruct	2013-02-16 13:19:02.480256783 +0100
@@ -3,7 +3,7 @@
 version = "1.1.1"
 
 #Replace @VERSION@ in certain files
-files = ["jngl.pc.in", "installer/mingw.nsi.in", 'installer/msvc.nsi.in', 'installer/python.nsi.in']
+files = ["jngl.pc.in"]
 for filename in files:
 	newfilename = filename.replace(".in", "")
 	datei = open(filename,"r").readlines()
