# add missing linker libs
Index: irrlamb-0.1.1/CMakeLists.txt
===================================================================
--- irrlamb-0.1.1.orig/CMakeLists.txt	2013-03-04 18:47:11.000000000 +0100
+++ irrlamb-0.1.1/CMakeLists.txt	2014-04-06 15:51:46.325603325 +0200
@@ -37,6 +37,7 @@
 find_package(Irrlicht 1.8 REQUIRED)
 find_package(OpenAL REQUIRED)
 find_package(Bullet REQUIRED)
+find_package(Threads REQUIRED)
 
 # include paths
 include_directories(${BULLET_INCLUDE_DIRS})
@@ -74,7 +75,7 @@
 add_executable(${CMAKE_PROJECT_NAME} ${SRC_ALL} src/resource.rc)
 
 # link libraries
-target_link_libraries(${CMAKE_PROJECT_NAME} ${IRRLICHT_LIBRARIES} ${BULLET_LIBRARIES} ${OPENAL_LIBRARY})
+target_link_libraries(${CMAKE_PROJECT_NAME} ${IRRLICHT_LIBRARIES} ${BULLET_LIBRARIES} ${OPENAL_LIBRARY} ${CMAKE_THREAD_LIBS_INIT} dl)
 
 # copy irrlicht DLLs to working directory
 if(WIN32)
