# lua 5.3 is not yet in the current Ubuntu releases
Index: irrlamb-0.2.0/CMakeLists.txt
===================================================================
--- irrlamb-0.2.0.orig/CMakeLists.txt	2015-09-20 14:55:39.526567379 +0200
+++ irrlamb-0.2.0/CMakeLists.txt	2015-09-20 15:04:57.747151097 +0200
@@ -43,9 +43,9 @@
 
 	# lua
 	set(Lua_FIND_VERSION_EXACT TRUE)
-	find_package(Lua 5.3 REQUIRED)
+	find_package(Lua 5.2 REQUIRED)
 	if(NOT LUA_FOUND)
-		message(SEND_ERROR "Could not find Lua 5.3!")
+		message(SEND_ERROR "Could not find Lua 5.2!")
 	endif()
 
 	if(PANDORA)
