Index: sauerbraten-0.0.20130203/Makefile
===================================================================
--- sauerbraten-0.0.20130203.orig/Makefile	2013-02-10 14:59:59.000000000 +0100
+++ sauerbraten-0.0.20130203/Makefile	2013-02-10 15:10:02.624409399 +0100
@@ -142,9 +142,9 @@
 	$(MAKE)	-C enet/ all
 
 clean-enet: enet/Makefile
-	$(MAKE) -C enet/ clean
+	$(MAKE) -C enet/ distclean
 
-clean:
+clean: clean-enet
 	-$(RM) $(CLIENT_PCH) $(CLIENT_OBJS) $(SERVER_OBJS) $(MASTER_OBJS) sauer_client sauer_server sauer_master
 
 %.h.gch: %.h
