# Use os.makedirs to create intermediate directories
Index: mypaint-1.2.0/gui/application.py
===================================================================
--- mypaint-1.2.0.orig/gui/application.py	2015-12-04 02:48:34.000000000 +0100
+++ mypaint-1.2.0/gui/application.py	2016-01-17 03:30:46.929802331 +0100
@@ -220,7 +220,7 @@
         # storage areas.
         for basedir in [state_dirs.user_config, state_dirs.user_data]:
             if not os.path.isdir(basedir):
-                os.mkdir(basedir)
+                os.makedirs(basedir)
                 logger.info('Created basedir %r', basedir)
         for datasubdir in [u'backgrounds', u'brushes', u'scratchpads']:
             datadir = os.path.join(state_dirs.user_data, datasubdir)
