From: Paul Wise <pabs@debian.org>
Date: Thu, 3 Mar 2011 09:22:38 +0800
Subject: Switch to using external miniupnpc
Index: warzone2100-2.3.9/configure.ac
===================================================================
--- warzone2100-2.3.9.orig/configure.ac	2011-10-23 22:28:21.000000000 +0200
+++ warzone2100-2.3.9/configure.ac	2011-11-06 20:04:38.013792413 +0100
@@ -416,6 +416,10 @@
 ])
 AM_CONDITIONAL([MOTIF_AVAILABLE], test -n "$MOTIF_LIBS")
 
+# Look for minupnpc
+AC_CHECK_HEADER(miniupnpc/miniupnpc.h, , AC_MSG_ERROR([miniupnpc header not found.]))
+AC_CHECK_LIB(miniupnpc, UPNP_GetExternalIPAddress, AC_SUBST([MINIUPNPC_LIBS], [-lminiupnpc]), AC_MSG_ERROR([miniupnpc not found.]), [${WIN32_LIBS}])
+
 # Look for Popt
 AC_CHECK_HEADER(popt.h, , AC_MSG_ERROR([Popt header not found.]))
 AC_CHECK_LIB(popt, poptGetContext, AC_SUBST([POPT_LIBS], [-lpopt]), AC_MSG_ERROR([Popt not found.]), [${WIN32_LIBS}])
@@ -501,7 +505,6 @@
 		lib/ivis_opengl/Makefile
 		lib/ivis_common/Makefile
 		lib/netplay/Makefile
-		lib/netplay/miniupnpc/Makefile
 		lib/script/Makefile
 		lib/sequence/Makefile
 		lib/sound/Makefile
Index: warzone2100-2.3.9/lib/netplay/Makefile.am
===================================================================
--- warzone2100-2.3.9.orig/lib/netplay/Makefile.am	2011-10-23 22:05:14.000000000 +0200
+++ warzone2100-2.3.9/lib/netplay/Makefile.am	2011-11-06 20:04:38.013792413 +0100
@@ -1,5 +1,3 @@
-SUBDIRS = miniupnpc
-
 AM_CPPFLAGS = $(SDL_CFLAGS) $(SDL_NET_CFLAGS) $(WZ_CPPFLAGS)
 AM_CFLAGS = $(WZ_CFLAGS)
 AM_CXXFLAGS = $(WZ_CXXFLAGS)
Index: warzone2100-2.3.9/lib/netplay/netplay.c
===================================================================
--- warzone2100-2.3.9.orig/lib/netplay/netplay.c	2011-10-23 22:28:21.000000000 +0200
+++ warzone2100-2.3.9/lib/netplay/netplay.c	2011-11-06 20:04:38.017792414 +0100
@@ -37,9 +37,9 @@
 #include "netplay.h"
 #include "netlog.h"
 
-#include "miniupnpc/miniwget.h"
-#include "miniupnpc/miniupnpc.h"
-#include "miniupnpc/upnpcommands.h"
+#include <miniupnpc/miniwget.h>
+#include <miniupnpc/miniupnpc.h>
+#include <miniupnpc/upnpcommands.h>
 #include "lib/exceptionhandler/dumpinfo.h"
 
 #include "src/multistat.h"
@@ -2126,9 +2126,9 @@
 	int r;
 
 	debug(LOG_NET, "upnp_add_redir(%d)\n", port);
-	UPNP_GetExternalIPAddress(urls.controlURL, data.servicetype, externalIP);
+	UPNP_GetExternalIPAddress(urls.controlURL, data.first.servicetype, externalIP);
 	sprintf(port_str, "%d", port);
-	r = UPNP_AddPortMapping(urls.controlURL, data.servicetype,
+	r = UPNP_AddPortMapping(urls.controlURL, data.first.servicetype,
 			port_str, port_str, lanaddr, "Warzone 2100", "TCP", 0);
 	if (r != UPNPCOMMAND_SUCCESS)
 	{
@@ -2144,7 +2144,7 @@
 	char port_str[16];
 	debug(LOG_NET, "upnp_rem_redir(%d)", port);
 	sprintf(port_str, "%d", port);
-	UPNP_DeletePortMapping(urls.controlURL, data.servicetype, port_str, "TCP", 0);
+	UPNP_DeletePortMapping(urls.controlURL, data.first.servicetype, port_str, "TCP", 0);
 }
 
 void NETaddRedirects(void)
Index: warzone2100-2.3.9/po/update-po.sh
===================================================================
--- warzone2100-2.3.9.orig/po/update-po.sh	2011-10-23 22:05:14.000000000 +0200
+++ warzone2100-2.3.9/po/update-po.sh	2011-11-06 20:04:38.017792414 +0100
@@ -11,5 +11,5 @@
 find lib src data -type f |
 	grep -v '\/.svn\/' |
 	grep -e '\.c\(pp\|xx\)\?$' -e 'data.*strings.*\.txt$' -e 'data.*sequenceaudio.*\.tx.$' -e '\.slo$' -e '\.rmsg$' |
-	grep -v -e '\.lex\.c\(pp\|xx\)\?$' -e '\.tab\.c\(pp\|xx\)\?$' -e 'lib/netplay/miniupnpc/*' -e 'GLee\.c' |
+	grep -v -e '\.lex\.c\(pp\|xx\)\?$' -e '\.tab\.c\(pp\|xx\)\?$' -e 'GLee\.c' |
 	sort >> po/POTFILES.in
Index: warzone2100-2.3.9/src/Makefile.am
===================================================================
--- warzone2100-2.3.9.orig/src/Makefile.am	2011-10-23 22:05:14.000000000 +0200
+++ warzone2100-2.3.9/src/Makefile.am	2011-11-06 20:04:38.017792414 +0100
@@ -8,7 +8,7 @@
 force-linker.cpp:
 	touch $@
 
-AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(OGGVORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(POPT_CFLAGS) $(WZ_CPPFLAGS) $(GLee_CFLAGS)
+AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(OGGVORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(POPT_CFLAGS) $(WZ_CPPFLAGS) $(GLee_CFLAGS) $(MINIUPNPC_LIBS)
 AM_CFLAGS = $(WZ_CFLAGS)
 AM_CXXFLAGS = $(WZ_CXXFLAGS)
 AM_LFLAGS = $(FLEX_FLAGS)
@@ -290,7 +290,6 @@
 	$(top_builddir)/lib/sound/libsound.a \
 	$(top_builddir)/lib/script/libscript.a \
 	$(top_builddir)/lib/netplay/libnetplay.a \
-	$(top_builddir)/lib/netplay/miniupnpc/libminiupnpc.a \
 	$(top_builddir)/lib/ivis_opengl/libivis_opengl.a \
 	$(top_builddir)/lib/ivis_common/libivis_common.a \
 	$(top_builddir)/lib/gamelib/libgamelib.a \
@@ -298,7 +297,7 @@
 	$(top_builddir)/lib/iniparser/libiniparser.a \
 	$(top_builddir)/lib/exceptionhandler/libexceptionhandler.a
 
-warzone2100_LDADD += $(LTLIBINTL) $(SDL_LIBS) $(PHYSFS_LIBS) $(PNG_LIBS) $(OGGVORBIS_LIBS) $(THEORA_LIBS) $(OPENAL_LIBS) $(OPENGLC_LIBS) $(OPENGL_LIBS) $(POPT_LIBS) $(MOTIF_LIBS) $(X11_LIBS) $(GLee_LIBS)
+warzone2100_LDADD += $(LTLIBINTL) $(SDL_LIBS) $(PHYSFS_LIBS) $(PNG_LIBS) $(OGGVORBIS_LIBS) $(THEORA_LIBS) $(OPENAL_LIBS) $(OPENGLC_LIBS) $(OPENGL_LIBS) $(POPT_LIBS) $(MOTIF_LIBS) $(X11_LIBS) $(GLee_LIBS) $(MINIUPNPC_LIBS)
 
 if MINGW32
 warzone2100_LDADD += $(top_builddir)/win32/warzone2100.o $(WIN32_LIBS)
