Author: Paul Wise <pabs@debian.org>
Description: use system libraries instead of embedded code copies
Forwarded: no
Index: warzone2100-3.2.1/configure.ac
===================================================================
--- warzone2100-3.2.1.orig/configure.ac	2016-07-29 07:10:18.621845901 +0200
+++ warzone2100-3.2.1/configure.ac	2016-07-29 07:10:18.621845901 +0200
@@ -399,6 +399,9 @@
 
 # Checks for modules:
 
+
+PKG_CHECK_MODULES([QUESOGLC], [quesoglc])
+
 if test "$backend" = "sdl"; then
 	PKG_CHECK_MODULES([SDL], [sdl2 >= 2.0.0])
 fi
@@ -475,8 +478,7 @@
 WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)"
 
 # 3rdparty libs
-WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)/3rdparty -I\$(top_srcdir)/3rdparty/quesoglc"
-
+WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)/3rdparty"
 
 AC_SUBST([WZ_CPPFLAGS], [${WZ_CPPFLAGS}])
 AC_SUBST([WZ_CFLAGS], [${WZ_CFLAGS}])
@@ -495,7 +497,6 @@
 		win32/Makefile
 		tests/Makefile
 		3rdparty/miniupnpc/Makefile
-		3rdparty/quesoglc/Makefile
 		lib/framework/Makefile
 		lib/exceptionhandler/Makefile
 		lib/gamelib/Makefile
Index: warzone2100-3.2.1/Makefile.am
===================================================================
--- warzone2100-3.2.1.orig/Makefile.am	2016-07-29 07:10:18.621845901 +0200
+++ warzone2100-3.2.1/Makefile.am	2016-07-29 07:10:18.621845901 +0200
@@ -7,7 +7,6 @@
 SUBDIRS = \
 	win32 \
 	3rdparty/miniupnpc \
-	3rdparty/quesoglc \
 	lib/framework \
 	lib/exceptionhandler \
 	lib/script \
Index: warzone2100-3.2.1/src/Makefile.am
===================================================================
--- warzone2100-3.2.1.orig/src/Makefile.am	2016-07-29 07:10:18.621845901 +0200
+++ warzone2100-3.2.1/src/Makefile.am	2016-07-29 07:10:18.621845901 +0200
@@ -1,4 +1,4 @@
-AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(VORBISFILE_CFLAGS) $(VORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(WZ_CPPFLAGS) $(GLEW_CFLAGS) $(LIBCRYPTO_CFLAGS)
+AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(VORBISFILE_CFLAGS) $(VORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(WZ_CPPFLAGS) $(GLEW_CFLAGS) $(LIBCRYPTO_CFLAGS) $(QUESOGLC_CFLAGS)
 AM_CFLAGS = $(WZ_CFLAGS)
 AM_CXXFLAGS = $(WZ_CXXFLAGS) $(QT5_CFLAGS)
 
@@ -297,8 +297,7 @@
 	$(top_builddir)/lib/ivis_opengl/libivis_opengl.a \
 	$(top_builddir)/lib/gamelib/libgamelib.a \
 	$(top_builddir)/lib/exceptionhandler/libexceptionhandler.a \
-	$(top_builddir)/3rdparty/miniupnpc/libminiupnpc.a \
-	$(top_builddir)/3rdparty/quesoglc/libquesoglc.a
+	$(top_builddir)/3rdparty/miniupnpc/libminiupnpc.a
 
 if PORTABLE
 warzone2100_portable_SOURCES = $(COMMONSOURCES) $(nodist_COMMONSOURCES)
@@ -343,6 +342,7 @@
 else
 warzone2100_LDADD = $(warzone2100_LIBS) $(LTLIBINTL) $(SDL_LIBS) $(PHYSFS_LIBS) $(PNG_LIBS) $(VORBISFILE_LIBS) $(VORBIS_LIBS) \
 	$(THEORA_LIBS) $(OPENAL_LIBS) $(OPENGLC_LIBS) $(OPENGL_LIBS) $(SQT5_LIBS) $(QT5_LIBS) $(GLEW_LIBS) \
+	$(QUESOGLC_LIBS) \
 	$(X_LIBS) $(X_EXTRA_LIBS) $(LIBCRYPTO_LIBS) $(LDFLAGS)
 endif
 
Index: warzone2100-3.2.1/tests/Makefile.am
===================================================================
--- warzone2100-3.2.1.orig/tests/Makefile.am	2016-07-29 07:10:18.621845901 +0200
+++ warzone2100-3.2.1/tests/Makefile.am	2016-07-29 07:12:17.147283840 +0200
@@ -22,9 +22,9 @@
 
 ivis_linktest_SOURCES = ivis_linktest.cpp
 ivis_linktest_LDADD = $(top_builddir)/lib/sdl/libsdl.a $(top_builddir)/lib/framework/libframework.a \
-	$(top_builddir)/lib/ivis_opengl/libivis_opengl.a $(top_builddir)/3rdparty/quesoglc/libquesoglc.a  \
+	$(top_builddir)/lib/ivis_opengl/libivis_opengl.a \
 	$(PHYSFS_LIBS) $(LIBCRYPTO_LIBS) $(QT5_LIBS) $(SDL_LIBS) $(OPENGL_LIBS) $(OPENGLC_LIBS) $(GLEW_LIBS) \
-	$(X_LIBS) $(X_EXTRA_LIBS) $(LDFLAGS) $(PNG_LIBS)
+	$(QUESOGLC_LIBS) $(X_LIBS) $(X_EXTRA_LIBS) $(LDFLAGS) $(PNG_LIBS)
 
 modeltest_SOURCES = modeltest.c
 
