Author: Paul Wise <pabs@debian.org>
Description: use system libraries instead of embedded code copies
Forwarded: no
Index: warzone2100-3.1.1/configure.ac
===================================================================
--- warzone2100-3.1.1.orig/configure.ac	2014-01-04 00:48:50.868740049 +0100
+++ warzone2100-3.1.1/configure.ac	2014-01-06 20:37:04.426663498 +0100
@@ -382,6 +382,8 @@
 
 # Checks for modules:
 
+PKG_CHECK_MODULES([QUESOGLC], [quesoglc])
+
 if test "$backend" = "sdl"; then
 	PKG_CHECK_MODULES([SDL], [sdl >= 1.2])
 fi
@@ -460,8 +462,7 @@
 WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)"
 
 # 3rdparty libs
-WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)/3rdparty -I\$(top_srcdir)/3rdparty/quesoglc"
-
+WZ_CPPFLAGS="${WZ_CPPFLAGS} -I\$(top_srcdir)/3rdparty"
 
 AC_SUBST([WZ_CPPFLAGS], [${WZ_CPPFLAGS}])
 AC_SUBST([WZ_CFLAGS], [${WZ_CFLAGS}])
@@ -480,7 +481,6 @@
 		win32/Makefile
 		tests/Makefile
 		3rdparty/miniupnpc/Makefile
-		3rdparty/quesoglc/Makefile
 		lib/framework/Makefile
 		lib/exceptionhandler/Makefile
 		lib/gamelib/Makefile
Index: warzone2100-3.1.1/Makefile.am
===================================================================
--- warzone2100-3.1.1.orig/Makefile.am	2014-01-04 00:48:50.868740049 +0100
+++ warzone2100-3.1.1/Makefile.am	2014-01-06 19:54:56.198738706 +0100
@@ -7,7 +7,6 @@
 SUBDIRS = \
 	win32 \
 	3rdparty/miniupnpc \
-	3rdparty/quesoglc \
 	lib/framework \
 	lib/exceptionhandler \
 	lib/script \
Index: warzone2100-3.1.1/src/Makefile.am
===================================================================
--- warzone2100-3.1.1.orig/src/Makefile.am	2014-01-04 00:48:50.868740049 +0100
+++ warzone2100-3.1.1/src/Makefile.am	2014-01-06 19:56:48.890735354 +0100
@@ -1,4 +1,4 @@
-AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(VORBISFILE_CFLAGS) $(VORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(WZ_CPPFLAGS) $(GLEW_CFLAGS)
+AM_CPPFLAGS = -DYY_NO_INPUT $(SDL_CFLAGS) $(PHYSFS_CFLAGS) $(PNG_CFLAGS) $(VORBISFILE_CFLAGS) $(VORBIS_CFLAGS) $(OPENAL_CFLAGS) $(OPENGLC_CFLAGS) $(OPENGL_CFLAGS) $(WZ_CPPFLAGS) $(GLEW_CFLAGS) $(QUESOGLC_CFLAGS)
 AM_CFLAGS = $(WZ_CFLAGS)
 AM_CXXFLAGS = $(WZ_CXXFLAGS) $(QT4_CFLAGS)
 
@@ -278,8 +278,7 @@
 	$(top_builddir)/lib/gamelib/libgamelib.a \
 	$(top_builddir)/lib/framework/libframework.a \
 	$(top_builddir)/lib/exceptionhandler/libexceptionhandler.a \
-	$(top_builddir)/3rdparty/miniupnpc/libminiupnpc.a \
-	$(top_builddir)/3rdparty/quesoglc/libquesoglc.a
+	$(top_builddir)/3rdparty/miniupnpc/libminiupnpc.a
 
 if PORTABLE
 warzone2100_portable_SOURCES = $(COMMONSOURCES) $(nodist_COMMONSOURCES)
@@ -313,6 +312,7 @@
 else
 warzone2100_LDADD = $(warzone2100_LIBS) $(LTLIBINTL) $(SDL_LIBS) $(PHYSFS_LIBS) $(PNG_LIBS) $(VORBISFILE_LIBS) $(VORBIS_LIBS) \
 	$(THEORA_LIBS) $(OPENAL_LIBS) $(OPENGLC_LIBS) $(OPENGL_LIBS) $(QT4_LIBS) $(GLEW_LIBS) \
+	$(QUESOGLC_LIBS) \
 	$(X_LIBS) $(X_EXTRA_LIBS) $(LDFLAGS)
 endif
 
