=== modified file 'client/gui-gtk-2.0/gui_main.c'
Index: freeciv-2.3.2/client/gui-gtk-2.0/gui_main.c
===================================================================
--- freeciv-2.3.2.orig/client/gui-gtk-2.0/gui_main.c	2012-03-31 14:35:49.000000000 +0200
+++ freeciv-2.3.2/client/gui-gtk-2.0/gui_main.c	2012-04-14 17:51:53.370527814 +0200
@@ -1520,6 +1520,10 @@
    * locale logic in gtk_init() causes problems with zh_CN (see PR#39475) */
   gtk_disable_setlocale();
 
+  /* Work around a bug which surfaced when a gtk+2.0 bug was fixed in 
+     2.24.6-0ubuntu3 https://launchpad.net/bugs/819780 */
+  g_setenv("GDK_NATIVE_WINDOWS", "1", TRUE);
+
   /* GTK withdraw gtk options. Process GTK arguments */
   gtk_init(&argc, &argv);
 
