Description: Patch to move from internal m4 to system (gnulib) m4
Author: Karl Goetz <karl@kgoetz.id.au>
Forwarded: no
Last-Update: <2010-06-27>

Index: freeciv-2.4.0/Makefile.am
===================================================================
--- freeciv-2.4.0.orig/Makefile.am	2013-09-14 23:03:30.881469483 +0200
+++ freeciv-2.4.0/Makefile.am	2013-09-14 23:03:30.869469483 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-ACLOCAL_AMFLAGS = -I m4 -I dependencies/m4
+ACLOCAL_AMFLAGS = -I m4 -I /usr/share/gnulib/m4
 
 if CLIENT
 CSUBDIRS = client
Index: freeciv-2.4.0/Makefile.in
===================================================================
--- freeciv-2.4.0.orig/Makefile.in	2013-09-14 23:03:30.881469483 +0200
+++ freeciv-2.4.0/Makefile.in	2013-09-14 23:05:17.713466305 +0200
@@ -45,8 +45,8 @@
 	bootstrap/config.sub bootstrap/depcomp bootstrap/install-sh \
 	bootstrap/ltmain.sh bootstrap/missing bootstrap/mkinstalldirs
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
-am__aclocal_m4_deps = $(top_srcdir)/dependencies/m4/mkstemp.m4 \
-	$(top_srcdir)/dependencies/m4/stdlib_h.m4 \
+am__aclocal_m4_deps = /usr/share/gnulib/m4/mkstemp.m4 \
+	/usr/share/gnulib/m4/stdlib_h.m4 \
 	$(top_srcdir)/m4/c99.m4 $(top_srcdir)/m4/codeset.m4 \
 	$(top_srcdir)/m4/compiler.m4 $(top_srcdir)/m4/debug.m4 \
 	$(top_srcdir)/m4/fcdb-mysql.m4 \
@@ -432,7 +432,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-ACLOCAL_AMFLAGS = -I m4 -I dependencies/m4
+ACLOCAL_AMFLAGS = -I m4 -I /usr/share/gnulib/m4
 @CLIENT_FALSE@CSUBDIRS = 
 @CLIENT_TRUE@CSUBDIRS = client
 @SERVER_FALSE@SSUBDIRS = 
Index: freeciv-2.4.0/aclocal.m4
===================================================================
--- freeciv-2.4.0.orig/aclocal.m4	2013-09-14 23:03:30.881469483 +0200
+++ freeciv-2.4.0/aclocal.m4	2013-09-14 23:06:17.681464521 +0200
@@ -9632,8 +9632,8 @@
 AC_SUBST([am__untar])
 ]) # _AM_PROG_TAR
 
-m4_include([dependencies/m4/mkstemp.m4])
-m4_include([dependencies/m4/stdlib_h.m4])
+m4_include([/usr/share/gnulib/m4/mkstemp.m4])
+m4_include([/usr/share/gnulib/m4/stdlib_h.m4])
 m4_include([m4/c99.m4])
 m4_include([m4/codeset.m4])
 m4_include([m4/compiler.m4])
Index: freeciv-2.4.0/autogen.sh
===================================================================
--- freeciv-2.4.0.orig/autogen.sh	2013-09-14 23:03:30.881469483 +0200
+++ freeciv-2.4.0/autogen.sh	2013-09-14 23:03:30.877469483 +0200
@@ -262,7 +262,7 @@
 fi
 
 echo "+ running $ACLOCAL ..."
-$ACLOCAL -I m4 -I dependencies/m4 $ACLOCAL_FLAGS || {
+$ACLOCAL -I m4 -I /usr/share/gnulib/m4 $ACLOCAL_FLAGS || {
   echo
   echo "$ACLOCAL failed - check that all needed development files are present on system"
   exit 1
