Name the executables according to the package name.
Index: minetest-delta-0.0.0+git20110719/src/CMakeLists.txt
===================================================================
--- minetest-delta-0.0.0+git20110719.orig/src/CMakeLists.txt	2011-07-19 20:05:09.086826638 +0200
+++ minetest-delta-0.0.0+git20110719/src/CMakeLists.txt	2011-07-19 20:06:51.006826672 +0200
@@ -141,9 +141,9 @@
 set(EXECUTABLE_OUTPUT_PATH ../bin)
 
 if(BUILD_CLIENT)
-	add_executable(minetest ${minetest_SRCS})
+	add_executable(minetest-delta ${minetest_SRCS})
 	target_link_libraries(
-		minetest
+		minetest-delta
 		${ZLIB_LIBRARIES}
 		${IRRLICHT_LIBRARY}
 		${OPENGL_LIBRARIES}
@@ -159,9 +159,9 @@
 endif(BUILD_CLIENT)
 
 if(BUILD_SERVER)
-	add_executable(minetestserver ${minetestserver_SRCS})
+	add_executable(minetestserver-delta ${minetestserver_SRCS})
 	target_link_libraries(
-		minetestserver
+		minetestserver-delta
 		${ZLIB_LIBRARIES}
 		${PLATFORM_LIBS}
 		jthread
@@ -216,7 +216,7 @@
 	endif()
 	
 	if(BUILD_SERVER)
-		set_target_properties(minetestserver PROPERTIES
+		set_target_properties(minetestserver-delta PROPERTIES
 				COMPILE_DEFINITIONS "SERVER")
 	endif(BUILD_SERVER)
 
@@ -233,7 +233,7 @@
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/../minetest.conf.example" DESTINATION ${EXAMPLE_CONF_DIR})
 
 if(BUILD_CLIENT)
-	install(TARGETS minetest DESTINATION ${BINDIR})
+	install(TARGETS minetest-delta DESTINATION ${BINDIR})
 
 	file(GLOB images "${CMAKE_CURRENT_SOURCE_DIR}/../data/*.png")
 
@@ -250,7 +250,7 @@
 endif(BUILD_CLIENT)
 
 if(BUILD_SERVER)
-	install(TARGETS minetestserver DESTINATION ${BINDIR})
+	install(TARGETS minetestserver-delta DESTINATION ${BINDIR})
 endif(BUILD_SERVER)
 
 # Subdirectories
