Just a small typo.
Index: minetest-delta-0.0.0+git20110719/src/tile.cpp
===================================================================
--- minetest-delta-0.0.0+git20110719.orig/src/tile.cpp	2011-07-19 19:51:26.296826363 +0200
+++ minetest-delta-0.0.0+git20110719/src/tile.cpp	2011-07-19 19:52:16.126826381 +0200
@@ -448,7 +448,7 @@
 	m_name_to_id.insert(name, id);
 
 	dstream<<"INFO: getTextureIdDirect(): name="<<name
-			<<": succesfully returning id="<<id<<std::endl;
+			<<": successfully returning id="<<id<<std::endl;
 	
 	return id;
 }
