Description: Fix for HuiJia adapter auto-config
Author: Richard Goedeken <Richard@fascinationsoftware.com>
Origin: upstream, https://bitbucket.org/richard42/mupen64plus-input-sdl/changeset/862053cf1d9e

---
diff --git a/data/InputAutoCfg.ini b/data/InputAutoCfg.ini
index 8ab3ce72a3c83dde245a1d204e38b62e1bd32f82..2c917fedfe7fc429681f08567f7f3d382b4bb26b 100644
--- a/data/InputAutoCfg.ini
+++ b/data/InputAutoCfg.ini
@@ -705,8 +705,8 @@ Start = button(9)
 Z Trig = button(8)
 B Button = button(2)
 A Button = button(1)
-C Button R = axis(5-)
-C Button L = axis(5+)
+C Button R = axis(3-)
+C Button L = axis(3+)
 C Button D = axis(2+)
 C Button U = axis(2-)
 R Trig = button(7)
@@ -729,16 +729,16 @@ Start = button(25)
 Z Trig = button(24)
 B Button = button(18)
 A Button = button(17)
-C Button R = axis(6-)
-C Button L = axis(6+)
-C Button D = axis(3+)
-C Button U = axis(3-)
+C Button R = axis(7-)
+C Button L = axis(7+)
+C Button D = axis(6+)
+C Button U = axis(6-)
 R Trig = button(23)
 L Trig = button(22)
 Mempak switch = key(44)
 Rumblepak switch = key(46)
 X Axis = axis(4-,4+)
-Y Axis = axis(7-,7+)
+Y Axis = axis(5-,5+)
 
 ; Asikgame - Super Dual Box (with two PS2 gamepads connected)
 ; this is a 2-port USB device. The Right port is controller #1, the Left port is #2
