Description: added auto-config for Hyperkin Tomee N64 controller
Author: Richard Goedeken <Richard@fascinationsoftware.com>
Origin: upstream, https://bitbucket.org/richard42/mupen64plus-input-sdl/changeset/18b3ab2408be

---
diff --git a/data/InputAutoCfg.ini b/data/InputAutoCfg.ini
index 2c917fedfe7fc429681f08567f7f3d382b4bb26b..152b36ef026781600696396497ac4ad28ebd3dff 100644
--- a/data/InputAutoCfg.ini
+++ b/data/InputAutoCfg.ini
@@ -359,6 +359,31 @@ Rumblepak switch = key(114)
 X Axis = axis(0-,0+)
 Y Axis = axis(1-,1+)
 
+[PC Game Controller]
+plugged = True
+plugin = 2
+mouse = False
+AnalogDeadzone = 4096,4096
+AnalogPeak = 32768,32768
+DPad R = hat(0 Right)
+DPad L = hat(0 Left)
+DPad D = hat(0 Down)
+DPad U = hat(0 Up)
+Start = button(9)
+Z Trig = button(8)
+B Button = button(4)
+A Button = button(5)
+C Button R = button(1)
+C Button L = button(3)
+C Button D = button(2)
+C Button U = button(0)
+R Trig = button(7)
+L Trig = button(6)
+Mempak switch = key(109)
+Rumblepak switch = key(114)
+X Axis = axis(0-,0+)
+Y Axis = axis(1-,1+)
+
 [SAITEK P880]
 plugged = True
 plugin = 2
