Index: oolite-1.77/GNUmakefile
===================================================================
--- oolite-1.77.orig/GNUmakefile	2013-09-06 20:14:59.199053362 +0200
+++ oolite-1.77/GNUmakefile	2013-09-06 20:14:59.195053362 +0200
@@ -41,9 +41,9 @@
     LIBJS = js_static
 
     ADDITIONAL_INCLUDE_DIRS      = -Isrc/SDL -Isrc/Core -Isrc/BSDCompat -Isrc/Core/Scripting -Isrc/Core/Materials -Isrc/Core/Entities -Isrc/Core/OXPVerifier -Isrc/Core/Debug -Isrc/Core/Tables
-    ADDITIONAL_OBJC_LIBS         = -lGLU -lGL -lX11 -lSDL -lSDL_mixer `js-config --libs` -lgnustep-base `nspr-config --libs` -lstdc++
-    ADDITIONAL_CFLAGS            = -Wall -DLINUX -DNEED_STRLCPY `sdl-config --cflags` `nspr-config --cflags` `js-config --cflags`
-    ADDITIONAL_OBJCFLAGS         = -Wall -std=c99 -DLOADSAVEGUI -DLINUX -DXP_UNIX -Wno-import `sdl-config --cflags` `nspr-config --cflags` `js-config --cflags`
+    ADDITIONAL_OBJC_LIBS         = -lGLU -lGL -lX11 -lSDL -lSDL_mixer `pkg-config --libs mozjs185` -lgnustep-base `nspr-config --libs` -lstdc++
+    ADDITIONAL_CFLAGS            = -Wall -DLINUX -DNEED_STRLCPY `sdl-config --cflags` `nspr-config --cflags` `pkg-config --cflags mozjs185`
+    ADDITIONAL_OBJCFLAGS         = -Wall -std=c99 -DLOADSAVEGUI -DLINUX -DXP_UNIX -Wno-import `sdl-config --cflags` `nspr-config --cflags` `pkg-config --cflags mozjs185`
     oolite_LIB_DIRS              += -L/usr/X11R6/lib/
     ifeq ($(use_deps),yes)
         oolite_LIB_DIRS          += -Ldeps/Linux-deps/$(HOST_ARCH)/lib_linker
