Index: gigi-freeorion-0.4.2~svn5890/test/CMakeLists.txt
===================================================================
--- gigi-freeorion-0.4.2~svn5890.orig/test/CMakeLists.txt	2012-08-17 02:13:44.000000000 +0200
+++ gigi-freeorion-0.4.2~svn5890/test/CMakeLists.txt	2013-03-23 00:24:59.284485950 +0100
@@ -45,7 +45,6 @@
 make_test_exec(AdamWriter)
 make_test_exec(EveParser)
 make_test_exec(EveWriter)
-make_test_exec(EveLayout)
 
 add_test_and_data_files(StrongIntegralTypedef)
 add_test_and_data_files(StrongSizeTypedef)
@@ -75,12 +74,3 @@
 foreach (test_file ${eve_test_files})
     add_test_and_data_files(EveWriter ${test_file})
 endforeach ()
-
-if (TARGET GiGiSDL)
-    file(GLOB pngs *.png)
-    file(COPY EveLayout_vs_Adobe_Begin.html begin_images ${pngs} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-
-    foreach (test_file ${eve_test_files})
-        add_test_and_data_files(EveLayout ${test_file} ${CMAKE_CURRENT_BINARY_DIR})
-    endforeach ()
-endif ()
