Index: gigi-freeorion-0.4.2~svn5890/cmake/Util.cmake
===================================================================
--- gigi-freeorion-0.4.2~svn5890.orig/cmake/Util.cmake	2012-08-17 02:13:44.000000000 +0200
+++ gigi-freeorion-0.4.2~svn5890/cmake/Util.cmake	2013-03-23 00:25:01.012485935 +0100
@@ -224,7 +224,7 @@
         COMPILE_FLAGS "${THIS_VARIANT_COMPILE_FLAGS}"
         LINK_FLAGS "${THIS_VARIANT_LINK_FLAGS}"
         LABELS "${PROJECT_NAME}"
-        # SOVERSION "${BOOST_VERSION}"
+        SOVERSION "0d"
         )
     endif (THIS_LIB_IS_STATIC)
 
