Description: Change preferences to meet Flightgear instllation on Debian
Forwarded: not-needed
Author: Christopher Baines <cbaines8@gmail.com>
 
Index: fgrun-1.6.1/fgrun.prefs
===================================================================
--- fgrun-1.6.1.orig/fgrun.prefs	2012-03-15 14:33:08.096200949 +0100
+++ fgrun-1.6.1/fgrun.prefs	2012-03-15 14:33:26.496200402 +0100
@@ -4,8 +4,9 @@
 
 [.]
 
-fg_exe:/usr/local/bin/fgfs
-fg_root:/usr/local/lib/FlightGear/data
-fg_scenery:/usr/local/lib/FlightGear/data/Scenery
+fg_exe:/usr/games/fgfs
+fg_root:/usr/share/games/flightgear
+fg_scenery:/usr/share/games/flightgear/Scenery
+ts_exe:/usr/games/terrasync
 aircraft:c172-3d
 airport:KSFO
