# https://bugzilla.gnome.org/show_bug.cgi?id=744309
Index: gnome-2048-3.18.2/configure.ac
===================================================================
--- gnome-2048-3.18.2.orig/configure.ac	2015-11-12 07:14:03.000000000 +0100
+++ gnome-2048-3.18.2/configure.ac	2016-01-01 03:25:09.025717423 +0100
@@ -4,7 +4,6 @@
 AM_INIT_AUTOMAKE([1.11 no-dist-gzip dist-xz foreign])
 AM_SILENT_RULES([yes])
 AM_MAINTAINER_MODE
-GNOME_MAINTAINER_MODE_DEFINES
 
 AM_PROG_VALAC([0.24.0])
 AM_PROG_CC_C_O
