To not conflict with the existing libirrlicht-dev (1.7) package
the headers are placed in a different directory.
Index: choria-0.4.1/FindIrrlicht.cmake
===================================================================
--- choria-0.4.1.orig/FindIrrlicht.cmake	2013-06-21 20:48:28.506245944 +0200
+++ choria-0.4.1/FindIrrlicht.cmake	2013-06-21 20:48:28.502245944 +0200
@@ -24,7 +24,7 @@
 		/usr/local
 	PATH_SUFFIXES
 		include
-		irrlicht
+		irrlicht1.8
 )
 
 # find debug library
