GL is not used.
Index: choria-0.3.2/SConstruct
===================================================================
--- choria-0.3.2.orig/SConstruct	2010-12-12 13:19:15.411294005 +0100
+++ choria-0.3.2/SConstruct	2010-12-12 13:19:17.871294003 +0100
@@ -19,7 +19,7 @@
 env.Append(LIBPATH="./libraries")
 
 # Libraries
-env.Append(LIBS=Split("GL GLU Irrlicht Xxf86vm Xext X11 pthread sqlite3"))
+env.Append(LIBS=Split("GLU Irrlicht Xxf86vm Xext X11 pthread sqlite3"))
 
 # Build the program
 choria = env.Program("choria", SOURCES)
