# do not create Cache dir in current dir
Index: gemrb-0.8.1/gemrb/core/Interface.cpp
===================================================================
--- gemrb-0.8.1.orig/gemrb/core/Interface.cpp	2014-05-05 21:14:04.701397222 +0200
+++ gemrb-0.8.1/gemrb/core/Interface.cpp	2014-05-05 21:26:15.693375477 +0200
@@ -1557,7 +1557,7 @@
 	CONFIG_PATH("GemRBPath", GemRBPath,
 				CopyGemDataPath(GemRBPath, _MAX_PATH));
 
-	CONFIG_PATH("CachePath", CachePath, "./Cache");
+	CONFIG_PATH("CachePath", CachePath, "/tmp");
 	FixPath( CachePath, false );
 
 	CONFIG_PATH("GUIScriptsPath", GUIScriptsPath, GemRBPath);
