# do not create Cache dir in current dir
Index: gemrb-0.8.2/gemrb/core/Interface.cpp
===================================================================
--- gemrb-0.8.2.orig/gemrb/core/Interface.cpp	2015-01-02 22:47:49.019731621 +0100
+++ gemrb-0.8.2/gemrb/core/Interface.cpp	2015-01-02 22:47:49.019731621 +0100
@@ -1556,7 +1556,7 @@
 	CONFIG_PATH("GemRBPath", GemRBPath,
 				CopyGemDataPath(GemRBPath, _MAX_PATH));
 
-	CONFIG_PATH("CachePath", CachePath, "./Cache");
+	CONFIG_PATH("CachePath", CachePath, "/tmp");
 	FixPath( CachePath, false );
 
 	CONFIG_PATH("GUIScriptsPath", GUIScriptsPath, GemRBPath);
