From: Markus Koschany <apo@gambaru.de>
Date: Wed, 20 Nov 2013 12:29:33 +0100
Subject: OISInput config

Forwarded: http://www.freeorion.org/forum/viewtopic.php?f=25&t=7719
---
 FreeOrion/client/human/chmain.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FreeOrion/client/human/chmain.cpp b/FreeOrion/client/human/chmain.cpp
index d2eb697..b08d9b5 100644
--- a/FreeOrion/client/human/chmain.cpp
+++ b/FreeOrion/client/human/chmain.cpp
@@ -284,7 +284,7 @@ int mainSetupAndRunOgre() {
         //EntityRenderer entity_renderer(scene_manager);
 
         parse::init();
-        HumanClientApp app(root, window, scene_manager, camera, viewport, GetRootDataDir() / "OISInput.cfg");
+        HumanClientApp app(root, window, scene_manager, camera, viewport, GetUserDir() / "OISInput.cfg");
 
 
         ois_input_plugin = new OISInput;
-- 
1.8.4.3

