Description: Fix data path for deb package build
Author: Fabio Fantoni <fabio.fantoni@tiscali.it>
Index: pokemon-online-2.4.1/src/Teambuilder/pluginmanager.cpp
===================================================================
--- pokemon-online-2.4.1.orig/src/Teambuilder/pluginmanager.cpp	2014-01-12 16:32:09.832376658 +0100
+++ pokemon-online-2.4.1/src/Teambuilder/pluginmanager.cpp	2014-01-12 16:46:51.056350444 +0100
@@ -133,5 +133,5 @@
 
 QString ClientPluginManager::directory() const
 {
-    return "clientplugins/";
+    return "/usr/lib/games/pokemon-online/clientplugins/";
 }
Index: pokemon-online-2.4.1/src/Server/Server.pro
===================================================================
--- pokemon-online-2.4.1.orig/src/Server/Server.pro	2014-01-12 16:32:09.832376658 +0100
+++ pokemon-online-2.4.1/src/Server/Server.pro	2014-01-12 16:33:41.152373942 +0100
@@ -6,7 +6,8 @@
     xml \
     sql
 
-TARGET = Server
+CONFIG(debian_package): TARGET = Pokemon-Online-Server
+!CONFIG(debian_package): TARGET = Server
 TEMPLATE = app
 SOURCES += main.cpp \
     consolereader.cpp \
Index: pokemon-online-2.4.1/src/BattleServer/pluginmanager.cpp
===================================================================
--- pokemon-online-2.4.1.orig/src/BattleServer/pluginmanager.cpp	2014-01-11 18:42:37.000000000 +0100
+++ pokemon-online-2.4.1/src/BattleServer/pluginmanager.cpp	2014-01-12 16:49:20.376346002 +0100
@@ -46,5 +46,5 @@
 
 QString BattleServerPluginManager::directory() const
 {
-    return "battleserverplugins/";
+    return "/usr/lib/games/pokemon-online/battleserverplugins/";
 }
Index: pokemon-online-2.4.1/src/Server/pluginmanager.cpp
===================================================================
--- pokemon-online-2.4.1.orig/src/Server/pluginmanager.cpp	2014-01-11 18:42:37.000000000 +0100
+++ pokemon-online-2.4.1/src/Server/pluginmanager.cpp	2014-01-12 16:49:23.784345901 +0100
@@ -20,5 +20,5 @@
 
 QString ServerPluginManager::directory() const
 {
-    return "serverplugins/";
+    return "/usr/lib/games/pokemon-online/serverplugins/";
 }
