Index: intensityengine-1.1.6/src/thirdparty/v8/src/globals.h
===================================================================
--- intensityengine-1.1.6.orig/src/thirdparty/v8/src/globals.h	2010-01-08 08:36:04.000000000 +0100
+++ intensityengine-1.1.6/src/thirdparty/v8/src/globals.h	2012-10-20 13:04:02.311836816 +0200
@@ -28,6 +28,8 @@
 #ifndef V8_GLOBALS_H_
 #define V8_GLOBALS_H_
 
+#include <string.h>
+
 namespace v8 {
 namespace internal {
 
