# use the datadir from configure directly.
Index: extremetuxracer-0.7.3/src/game_config.cpp
===================================================================
--- extremetuxracer-0.7.3.orig/src/game_config.cpp	2016-07-23 19:59:10.980596508 +0200
+++ extremetuxracer-0.7.3/src/game_config.cpp	2016-07-23 19:59:10.976596425 +0200
@@ -304,8 +304,8 @@
 		mkdir(param.config_dir.c_str(), 0775);
         }
 	param.data_dir = ETR_DATA_DIR;
-	param.data_dir += SEP;
-	param.data_dir += "etr";
+	/*param.data_dir += SEP;
+	param.data_dir += "etr";*/
         param.save_dir = param.config_dir;
 	param.configfile = param.config_dir + SEP "options.txt";
 	if (FileExists(param.configfile)) {
