Always include curses.h. Else functions are not found.
Index: stone-soup-0.16.1/source/libunix.cc
===================================================================
--- stone-soup-0.16.1.orig/source/libunix.cc	2015-06-06 19:55:35.405528445 +0200
+++ stone-soup-0.16.1/source/libunix.cc	2015-06-06 19:55:35.401528385 +0200
@@ -45,16 +45,7 @@
 
 #include <time.h>
 
-// Its best if curses comes at the end (name conflicts with Solaris). -- bwr
-#ifndef CURSES_INCLUDE_FILE
-    #ifndef _XOPEN_SOURCE_EXTENDED
-    #define _XOPEN_SOURCE_EXTENDED
-    #endif
-
-    #include <curses.h>
-#else
-    #include CURSES_INCLUDE_FILE
-#endif
+#include <curses.h>
 
 // Globals holding current text/backg. colours
 static short FG_COL = WHITE;
