contrib directory does not exist.
Index: stone-soup-0.17.1/source/Makefile
===================================================================
--- stone-soup-0.17.1.orig/source/Makefile	2015-12-10 20:42:37.401494352 +0100
+++ stone-soup-0.17.1/source/Makefile	2015-12-10 20:42:37.401494352 +0100
@@ -1503,7 +1503,6 @@
 	$(RM) -r build
 
 clean-contrib:
-	+$(MAKE) -C contrib clean
 
 clean-android:
 ifneq ($(ANDROID),)
