contrib directory does not exist.
Index: stone-soup-0.19.5/source/Makefile
===================================================================
--- stone-soup-0.19.5.orig/source/Makefile	2017-03-01 19:17:59.619543054 +0100
+++ stone-soup-0.19.5/source/Makefile	2017-03-01 19:17:59.615543103 +0100
@@ -1446,7 +1446,6 @@
 	$(RM) -r build
 
 clean-contrib:
-	+$(MAKE) -C contrib clean
 
 clean-android:
 ifneq ($(ANDROID),)
