contrib directory does not exist.
Index: stone-soup-0.12.0/source/Makefile
===================================================================
--- stone-soup-0.12.0.orig/source/Makefile	2013-04-29 22:00:49.000000000 +0200
+++ stone-soup-0.12.0/source/Makefile	2013-05-02 22:32:24.243904302 +0200
@@ -1351,7 +1351,6 @@
 	$(RM) -r build
 
 clean-contrib:
-	+$(MAKE) -C contrib clean
 
 clean-android:
 	$(RM) ../icon.png ../libapplication.so
