contrib directory does not exist.
Index: stone-soup-0.17.0/source/Makefile
===================================================================
--- stone-soup-0.17.0.orig/source/Makefile	2015-11-06 20:14:26.559041765 +0100
+++ stone-soup-0.17.0/source/Makefile	2015-11-06 20:14:26.559041765 +0100
@@ -1506,7 +1506,6 @@
 	$(RM) -r build
 
 clean-contrib:
-	+$(MAKE) -C contrib clean
 
 clean-android:
 ifneq ($(ANDROID),)
