contrib directory does not exist.
Index: stone-soup-0.16.1/source/Makefile
===================================================================
--- stone-soup-0.16.1.orig/source/Makefile	2015-06-06 19:55:42.833739502 +0200
+++ stone-soup-0.16.1/source/Makefile	2015-06-06 19:55:42.833739502 +0200
@@ -1489,7 +1489,6 @@
 	$(RM) -r build
 
 clean-contrib:
-	+$(MAKE) -C contrib clean
 
 clean-android:
 ifneq ($(ANDROID),)
