Just a typo.
Index: stone-soup-0.16.1/source/package.cc
===================================================================
--- stone-soup-0.16.1.orig/source/package.cc	2015-06-06 19:55:45.077841549 +0200
+++ stone-soup-0.16.1/source/package.cc	2015-06-06 19:55:45.073841346 +0200
@@ -85,7 +85,6 @@
 #endif
 {
     dprintf("package: initializing file=\"%s\" rw=%d\n", file, writeable);
-    ASSERT(writeable || !empty);
     filename = file;
     rw = writeable;
 
