Just a typo.
Index: stone-soup-0.13.0/source/package.cc
===================================================================
--- stone-soup-0.13.0.orig/source/package.cc	2013-11-02 15:01:37.319400346 +0100
+++ stone-soup-0.13.0/source/package.cc	2013-11-02 15:01:37.311400346 +0100
@@ -85,7 +85,6 @@
   : n_users(0), dirty(false), aborted(false), tmp(false)
 {
     dprintf("package: initializing file=\"%s\" rw=%d\n", file, writeable);
-    ASSERT(writeable || !empty);
     filename = file;
     rw = writeable;
 
