Just a typo.
Index: stone-soup-0.12.0/source/package.cc
===================================================================
--- stone-soup-0.12.0.orig/source/package.cc	2013-04-29 22:00:52.000000000 +0200
+++ stone-soup-0.12.0/source/package.cc	2013-05-02 22:32:28.475904175 +0200
@@ -90,7 +90,6 @@
   : n_users(0), dirty(false), aborted(false), tmp(false)
 {
     dprintf("package: initializing file=\"%s\" rw=%d\n", file, writeable);
-    ASSERT(writeable || !empty);
     filename = file;
     rw = writeable;
 
