# hardcode path to database and translations
Index: evernus-1.29/EvernusApplication.cpp
===================================================================
--- evernus-1.29.orig/EvernusApplication.cpp	2015-04-08 21:41:16.996536131 +0200
+++ evernus-1.29/EvernusApplication.cpp	2015-04-08 21:42:36.684533760 +0200
@@ -1615,11 +1615,11 @@
         removeTranslator(&mTranslator);
         removeTranslator(&mQtTranslator);
 
-        mTranslator.load(locale, "lang", "_", applicationDirPath() + UISettings::translationPath);
-        mQtTranslator.load(locale, "qt", "_", applicationDirPath() + UISettings::translationPath);
-        mQtBaseTranslator.load(locale, "qtbase", "_", applicationDirPath() + UISettings::translationPath);
-        mQtScriptTranslator.load(locale, "qtscript", "_", applicationDirPath() + UISettings::translationPath);
-        mQtXmlPatternsTranslator.load(locale, "qtxmlpatterns", "_", applicationDirPath() + UISettings::translationPath);
+        mTranslator.load(locale, "lang", "_", "/usr/share/games/evernus/trans/");
+        mQtTranslator.load(locale, "qt", "_", "/usr/share/games/evernus/trans/");
+        mQtBaseTranslator.load(locale, "qtbase", "_", "/usr/share/games/evernus/trans/");
+        mQtScriptTranslator.load(locale, "qtscript", "_", "/usr/share/games/evernus/trans/");
+        mQtXmlPatternsTranslator.load(locale, "qtxmlpatterns", "_", "/usr/share/games/evernus/trans/");
 
         installTranslator(&mQtXmlPatternsTranslator);
         installTranslator(&mQtScriptTranslator);
@@ -1635,7 +1635,7 @@
         if (!mEveDb.isValid())
             throw std::runtime_error{"Error crating Eve DB object!"};
 
-        auto eveDbPath = applicationDirPath() + "/res/eve.db";
+        QString eveDbPath = "/usr/share/games/evernus/res/eve.db";
         if (!QFile::exists(eveDbPath))
         {
             eveDbPath = QStandardPaths::locate(QStandardPaths::GenericDataLocation, applicationName() + "/res/eve.db");
