# hardcode path to database and translations
Index: evernus-1.42/EvernusApplication.cpp
===================================================================
--- evernus-1.42.orig/EvernusApplication.cpp	2016-06-15 19:38:40.759765154 +0200
+++ evernus-1.42/EvernusApplication.cpp	2016-06-15 19:38:40.755765057 +0200
@@ -1755,11 +1755,11 @@
         removeTranslator(&mTranslator);
         removeTranslator(&mQtTranslator);
 
-        mTranslator.load(locale, "lang", "_", applicationDirPath() + UISettings::translationPath);
-        mQtTranslator.load(locale, "qt", "_", applicationDirPath() + UISettings::translationPath);
-        mQtBaseTranslator.load(locale, "qtbase", "_", applicationDirPath() + UISettings::translationPath);
-        mQtScriptTranslator.load(locale, "qtscript", "_", applicationDirPath() + UISettings::translationPath);
-        mQtXmlPatternsTranslator.load(locale, "qtxmlpatterns", "_", applicationDirPath() + UISettings::translationPath);
+        mTranslator.load(locale, "lang", "_", "/usr/share/games/evernus/trans/");
+        mQtTranslator.load(locale, "qt", "_", "/usr/share/games/evernus/trans/");
+        mQtBaseTranslator.load(locale, "qtbase", "_", "/usr/share/games/evernus/trans/");
+        mQtScriptTranslator.load(locale, "qtscript", "_", "/usr/share/games/evernus/trans/");
+        mQtXmlPatternsTranslator.load(locale, "qtxmlpatterns", "_", "/usr/share/games/evernus/trans/");
 
         installTranslator(&mQtXmlPatternsTranslator);
         installTranslator(&mQtScriptTranslator);
@@ -1775,7 +1775,7 @@
         if (!mEveDb.isValid())
             throw std::runtime_error{"Error crating Eve DB object!"};
 
-        auto eveDbPath = applicationDirPath() + "/res/eve.db";
+        QString eveDbPath = "/usr/share/games/evernus/res/eve.db";
         if (!QFile::exists(eveDbPath))
         {
             eveDbPath = QStandardPaths::locate(QStandardPaths::GenericDataLocation, applicationName() + "/res/eve.db");
