Do not include directories which are in other packages.
Index: speed-dreams-base-2.0.0/data/cars/models/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/data/cars/models/CMakeLists.txt	2012-05-14 18:53:16.172495555 +0200
+++ speed-dreams-base-2.0.0/data/cars/models/CMakeLists.txt	2012-05-14 18:54:10.228493947 +0200
@@ -1,24 +1,6 @@
 INCLUDE(../../../cmake/macros.cmake)
 
 # Official car sets.
-ADD_SUBDIRECTORY(36gp-century-r11b)
-ADD_SUBDIRECTORY(36gp-ettore-t59)
-ADD_SUBDIRECTORY(36gp-milano-12c36)
-ADD_SUBDIRECTORY(36gp-milano-tipop3)
-ADD_SUBDIRECTORY(36gp-motorunion-typec)
-ADD_SUBDIRECTORY(36gp-silber-w25b)
-ADD_SUBDIRECTORY(36gp-tridenti-6c34)
-ADD_SUBDIRECTORY(36gp-tridenti-v8ri)
-
-ADD_SUBDIRECTORY(trb1-boxer-trbrs)
-ADD_SUBDIRECTORY(trb1-cavallo-360rb)
-ADD_SUBDIRECTORY(trb1-sector-rb1)
-ADD_SUBDIRECTORY(trb1-silber-rblk)
-ADD_SUBDIRECTORY(trb1-spirit-rb1lt)
-ADD_SUBDIRECTORY(trb1-taipan-rb1)
-ADD_SUBDIRECTORY(trb1-vieringe-5rb/)
-ADD_SUBDIRECTORY(trb1-zaxxon-rb)
-
 ADD_SUBDIRECTORY(sc-boxer-96)
 ADD_SUBDIRECTORY(sc-cavallo-360)
 ADD_SUBDIRECTORY(sc-fmc-gt4)
@@ -26,32 +8,6 @@
 ADD_SUBDIRECTORY(sc-murasama-nsx)
 ADD_SUBDIRECTORY(sc-spirit-300)
 
-ADD_SUBDIRECTORY(ls1-archer-r9)
-ADD_SUBDIRECTORY(ls1-newcastle-fury)
-ADD_SUBDIRECTORY(ls1-taipan-ltsr)
-ADD_SUBDIRECTORY(ls1-toro-rgt)
-ADD_SUBDIRECTORY(ls1-vulture-v6r)
-ADD_SUBDIRECTORY(ls1-zentek-z7r)
-
-ADD_SUBDIRECTORY(ls2-bavaria-g3gtr)
-ADD_SUBDIRECTORY(ls2-boxer-96rsr)
-ADD_SUBDIRECTORY(ls2-cavallo-360r)
-ADD_SUBDIRECTORY(ls2-condor-c400r)
-ADD_SUBDIRECTORY(ls2-morley-r900)
-ADD_SUBDIRECTORY(ls2-sprite-gts)
-ADD_SUBDIRECTORY(ls2-vortex-gt)
-ADD_SUBDIRECTORY(ls2-lynx-xkr)
-
-ADD_SUBDIRECTORY(mp5-fmc)
-ADD_SUBDIRECTORY(mp5-fmc-drift)
-ADD_SUBDIRECTORY(mp5-fmc-speedy)
-
-ADD_SUBDIRECTORY(rs1-citrus-cs4)
-ADD_SUBDIRECTORY(rs1-fmc-centre)
-ADD_SUBDIRECTORY(rs1-kenji-08)
-ADD_SUBDIRECTORY(rs2-fai-dot)
-ADD_SUBDIRECTORY(rs2-lion-2m)
-
 # Work-in-progress car sets.
 IF(NOT OPTION_OFFICIAL_ONLY)
 
Index: speed-dreams-base-2.0.0/data/tracks/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/data/tracks/CMakeLists.txt	2012-05-14 18:53:16.184495553 +0200
+++ speed-dreams-base-2.0.0/data/tracks/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -1,10 +1,7 @@
 INCLUDE(../../cmake/macros.cmake)
 
 # Official track sets.
-ADD_SUBDIRECTORY(dirt)
 ADD_SUBDIRECTORY(circuit)
-ADD_SUBDIRECTORY(speedway)
-ADD_SUBDIRECTORY(road)
 
 # Work-in-progress track sets.
 IF(NOT OPTION_OFFICIAL_ONLY)
Index: speed-dreams-base-2.0.0/src/drivers/simplix/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/src/drivers/simplix/CMakeLists.txt	2012-05-14 18:53:16.244495553 +0200
+++ speed-dreams-base-2.0.0/src/drivers/simplix/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -5,12 +5,7 @@
 
 # Official-only simplix interface.
 SET(ROBOT_INTERFACE LEGACY WELCOME
-                    simplix_trb1a simplix_trb1b 
                     simplix_sca simplix_scb 
-                    simplix_36GP 
-                    simplix_ls1 
-                    simplix_ls2 
-                    simplix_mp5 
                     my_simplix_0 my_simplix_1 my_simplix_2 my_simplix_3 my_simplix_4
                     my_simplix_5 my_simplix_6 my_simplix_7 my_simplix_8 my_simplix_9)
 
@@ -83,7 +78,7 @@
 )
 
 # Official-only simplix instances.
-SET(ROBOT_CLONES simplix_36GP simplix_sc simplix_trb1 simplix_ls1 simplix_ls2 simplix_mp5)
+SET(ROBOT_CLONES simplix_sc)
 
 IF(NOT OPTION_OFFICIAL_ONLY)
 
@@ -100,10 +95,7 @@
 
 # The data associated to the ubiquitous robot module and its clones.
 # simplix
-SET(ROBOT_SUBDIRS trb1-cavallo-360rb
-                  ls1-archer-r9 ls1-cavallo-570s1 ls1-newcastle-fury ls1-taipan-ltsr
-                  ls1-toro-rgt ls1-vulture-v6r ls1-zentek-z7r
-                  sc-boxer-96 sc-cavallo-360 sc-fmc-gt4 sc-lynx-220 
+SET(ROBOT_SUBDIRS sc-boxer-96 sc-cavallo-360 sc-fmc-gt4 sc-lynx-220 
                   sc-murasama-nsx sc-spirit-300
                   tracks
                   0)
@@ -112,16 +104,6 @@
            FILES ${ROBOT_NAME}.xml default.xml logo.rgb readme.txt
            SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml *.rgb)
 
-# 36GP
-SET(ROBOT_SUBDIRS 0 1 2 3 4 5 6 7 8 9
-                  36gp-ettore-t59 36gp-milano-12c36 36gp-silber-w25b 
-                  36gp-tridenti-6c34 36gp-tridenti-v8ri
-                  tracks)
-
-ROBOT_DATA(NAME simplix_36GP PREFIX simplix_36GP 
-           FILES simplix_36GP.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
 # SuperCars
 SET(ROBOT_SUBDIRS 0 1 2 3 4 5 6 7 8 9 12 13
                   sc-boxer-96 sc-spirit-300 sc-cavallo-360 sc-fmc-gt4 
@@ -132,45 +114,6 @@
            FILES simplix_sc.xml default.xml logo.rgb readme.txt
            SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
 
-# LS1 
-SET(ROBOT_SUBDIRS 0 1 2 3 4 5 6 7 8 9 10 11 12 13
-                  ls1-archer-r9 ls1-vulture-v6r ls1-toro-rgt
-                  ls1-cavallo-570s1 ls1-zentek-z7r ls1-newcastle-fury
-			ls1-taipan-ltsr
-			tracks)
-
-ROBOT_DATA(NAME simplix_ls1 PREFIX simplix_ls1 
-           FILES simplix_ls1.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-# LS2 
-SET(ROBOT_SUBDIRS 0
-                  ls2-bavaria-g3gtr
-			tracks)
-
-ROBOT_DATA(NAME simplix_ls2 PREFIX simplix_ls2 
-           FILES simplix_ls2.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-# TRB1
-SET(ROBOT_SUBDIRS 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
-                  trb1-cavallo-360rb trb1-boxer-trbrs trb1-spirit-rb1lt
-			trb1-taipan-rb1 trb1-silber-rblk trb1-sector-rb1
-                  trb1-zaxxon-rb trb1-vieringe-5rb
-                  tracks)
-
-ROBOT_DATA(NAME simplix_trb1 PREFIX simplix_trb1 
-           FILES simplix_trb1.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-# MP5
-SET(ROBOT_SUBDIRS 0 1 2 3 4 5 
-                  mp5-fmc mp5-fmc-drift mp5-fmc-speedy
-                  tracks)
-
-ROBOT_DATA(NAME simplix_mp5 PREFIX simplix_mp5 
-           FILES simplix_mp5.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
 # Work-in-progress / unofficial simplix instances.
 IF(NOT OPTION_OFFICIAL_ONLY)
 
Index: speed-dreams-base-2.0.0/src/drivers/usr/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/src/drivers/usr/CMakeLists.txt	2012-05-14 18:53:16.260495551 +0200
+++ speed-dreams-base-2.0.0/src/drivers/usr/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -6,12 +6,7 @@
 
 # Official-only USR interface.
 SET(ROBOT_INTERFACE LEGACY WELCOME
-                    usr_trb1
                     usr_sc
-                    usr_ls1
-                    usr_ls2
-                    usr_36GP
-                    usr_rs
                     )
 
 SET(ROBOT_SOURCES
@@ -36,7 +31,7 @@
   src/xmldefs.h)
 
 # Official-only USR instances.
-SET(ROBOT_CLONES usr_trb1 usr_sc usr_ls1 usr_ls2 usr_36GP usr_rs)
+SET(ROBOT_CLONES usr_sc)
 
 ROBOT_MODULE(NAME ${ROBOT_NAME} VERSION 3.0.0 SOVERSION 1.0.0
               INTERFACE ${ROBOT_INTERFACE} 
@@ -54,20 +49,6 @@
            #FILES README
            SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png)
 
-
-###################
-# USR_TRB1
-###################
-SET(ROBOT_SUBDIRS 1 2 3 4 5 6 7 8 9 10 11 12 13 14
-                  trb1-boxer-trbrs trb1-cavallo-360rb trb1-sector-rb1
-                  trb1-silber-rblk trb1-spirit-rb1lt trb1-taipan-rb1 
-                  trb1-zaxxon-rb
-                  tracks)
-
-ROBOT_DATA(NAME usr_trb1 PREFIX usr_trb1
-           FILES usr_trb1.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
 ###################
 # USR_SC
 ###################
@@ -79,55 +60,3 @@
 ROBOT_DATA(NAME usr_sc PREFIX usr_sc
            FILES usr_sc.xml default.xml logo.rgb readme.txt
            SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-###################
-# USR_LS1
-###################
-SET(ROBOT_SUBDIRS 1 2 3 4 5 6 7 8 9 10 11 12 13 14
-                  ls1-archer-r9 ls1-vulture-v6r ls1-zentek-z7r
-                  ls1-cavallo-570s1 ls1-newcastle-fury
-                  ls1-taipan-ltsr ls1-toro-rgt
-                  tracks)
-
-ROBOT_DATA(NAME usr_ls1 PREFIX usr_ls1
-           FILES usr_ls1.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-###################
-# USR_LS2
-###################
-SET(ROBOT_SUBDIRS 1 2 3 4 5 6 7 8 9 10 11 12 13 14
-                  ls2-bavaria-g3gtr ls2-boxer-96rsr
-                  ls2-cavallo-360r ls2-condor-c400r
-		  ls2-morley-r900 ls2-sprite-gts
-		  ls2-vortex-gt ls2-lynx-xkr
-                  tracks)
-
-ROBOT_DATA(NAME usr_ls2 PREFIX usr_ls2
-           FILES usr_ls2.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-###################
-# USR_36GP
-###################
-SET(ROBOT_SUBDIRS 1 2 3 4 5 6 7 8 9 10 11 12
-                  36gp-ettore-t59 36gp-milano-12c36 36gp-milano-tipop3
-                  36gp-motorunion-typec 36gp-tridenti-6c34
-                  36gp-tridenti-v8ri
-                  tracks)
-
-ROBOT_DATA(NAME usr_36GP PREFIX usr_36GP
-           FILES usr_36GP.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
-###################
-# USR_RS
-###################
-SET(ROBOT_SUBDIRS 1
-                  rs1-kenji-08
-                  tracks)
-
-ROBOT_DATA(NAME usr_rs PREFIX usr_rs
-           FILES usr_rs.xml default.xml logo.rgb readme.txt
-           SUBDIRS ${ROBOT_SUBDIRS} PATTERNS *.xml logo.rgb *.png *.jpg)
-
Index: speed-dreams-base-2.0.0/src/modules/simu/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/src/modules/simu/CMakeLists.txt	2012-05-14 18:53:16.216495553 +0200
+++ speed-dreams-base-2.0.0/src/modules/simu/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -1,10 +1,6 @@
 INCLUDE(../../../cmake/macros.cmake)
 
 ADD_SUBDIRECTORY(simuv2.1)
-ADD_SUBDIRECTORY(simuv3)
-
-# Should normally becomes obsolete in 2.0.
-ADD_SUBDIRECTORY(simuv2)
 
 # Unofficial / obsolete simulation engines.
 IF(NOT OPTION_OFFICIAL_ONLY)
Index: speed-dreams-base-2.0.0/src/raceman/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/src/raceman/CMakeLists.txt	2012-05-14 18:53:16.228495552 +0200
+++ speed-dreams-base-2.0.0/src/raceman/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -1,14 +1,7 @@
 INCLUDE(../../cmake/macros.cmake)
 
 SD_INSTALL_FILES(DATA config/raceman USER config/raceman 
-                 FILES practice.xml quickrace.xml
-                       singleevent-36gp.xml
-                       singleevent-challenge.xml singleevent-endurance.xml
-		       singleevent-ls-gt1.xml singleevent-mp5.xml
-                       championship.xml championship-supercars.xml championship-trb1.xml
-                       championship-36gp.xml championship-ls-gt1.xml championship-mp5.xml
-                       career.xml 
-                       career_supercars.xmls career_ls1.xmls)
+                 FILES singleevent-challenge.xml singleevent-endurance.xml)
 
 SD_INSTALL_FILES(DATA config/raceman/extra USER config/raceman/extra 
                  FILES extra/skill.xml)
Index: speed-dreams-base-2.0.0/data/tracks/circuit/CMakeLists.txt
===================================================================
--- speed-dreams-base-2.0.0.orig/data/tracks/circuit/CMakeLists.txt	2012-05-14 18:53:16.200495554 +0200
+++ speed-dreams-base-2.0.0/data/tracks/circuit/CMakeLists.txt	2012-05-14 18:54:10.232493947 +0200
@@ -1,16 +1,7 @@
 INCLUDE(../../../cmake/macros.cmake)
 
 # Official track sets.
-ADD_SUBDIRECTORY(aalborg)
-ADD_SUBDIRECTORY(brondehach)
-ADD_SUBDIRECTORY(chemisay)
-ADD_SUBDIRECTORY(corkscrew)
 ADD_SUBDIRECTORY(espie)
-ADD_SUBDIRECTORY(forza)
-ADD_SUBDIRECTORY(goldstone-sand)
-ADD_SUBDIRECTORY(karwada)
-ADD_SUBDIRECTORY(migrants)
-ADD_SUBDIRECTORY(ruudskogen)
 
 # Work-in-progress track sets.
 IF(NOT OPTION_OFFICIAL_ONLY)
