# add missing file
# change data directory
Index: cdogs-sdl-0.6.5/INSTALL
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ cdogs-sdl-0.6.5/INSTALL	2017-04-23 13:32:22.505792841 +0200
@@ -0,0 +1 @@
+#empty
Index: cdogs-sdl-0.6.5/CMakeLists.txt
===================================================================
--- cdogs-sdl-0.6.5.orig/CMakeLists.txt	2017-04-23 13:32:22.509792900 +0200
+++ cdogs-sdl-0.6.5/CMakeLists.txt	2017-04-23 13:34:06.706637926 +0200
@@ -185,16 +185,6 @@
 else()
 	set(LIB_EXTENSION ".so")
 endif()
-install(
-  PROGRAMS
-    ${CMAKE_CURRENT_BINARY_DIR}/src/cdogs-sdl${EXE_EXTENSION}
-  DESTINATION ${INSTALL_PREFIX}/bin)
-if(NOT "${GCW0}")
-	install(
-	  PROGRAMS
-	    ${CMAKE_CURRENT_BINARY_DIR}/src/cdogs-sdl-editor${EXE_EXTENSION}
-	  DESTINATION ${INSTALL_PREFIX}/bin)
-endif()
 
 INSTALL(DIRECTORY
 	${CMAKE_SOURCE_DIR}/doc
@@ -205,9 +195,6 @@
 	${CMAKE_SOURCE_DIR}/music
 	${CMAKE_SOURCE_DIR}/sounds
 	DESTINATION ${DATA_INSTALL_DIR})
-INSTALL(FILES
-	${CMAKE_SOURCE_DIR}/README.md
-	DESTINATION ${DATA_INSTALL_DIR})
 IF(UNIX AND NOT APPLE AND NOT BEOS AND NOT HAIKU)
 	INSTALL(FILES ${CMAKE_SOURCE_DIR}/build/linux/cdogs-sdl.desktop DESTINATION ${INSTALL_PREFIX}/share/applications)
 	INSTALL(FILES ${CMAKE_SOURCE_DIR}/build/linux/cdogs-sdl.appdata.xml DESTINATION ${INSTALL_PREFIX}/share/appdata)
