Don't try to call a script which does not exist.
Index: ufoai-2.4/build/maps.mk
===================================================================
--- ufoai-2.4.orig/build/maps.mk	2012-04-24 19:42:59.203662140 +0200
+++ ufoai-2.4/build/maps.mk	2012-04-24 19:43:07.215661903 +0200
@@ -24,10 +24,10 @@
 
 # TODO only sync if there were updates on the map files
 maps-sync:
-	$(PYTHONBIN) contrib/map-get/update.py
+#	$(PYTHONBIN) contrib/map-get/update.py
 
 force-maps-sync:
-	$(PYTHONBIN) contrib/map-get/update.py --reply=yes
+#	$(PYTHONBIN) contrib/map-get/update.py --reply=yes
 
 clean-maps:
 	@echo "Deleting maps..."
