# find python in multi-arch lib dir
Index: vegastrike-0.5.1~r1/m4scripts/VS_CHECK_PYTHON.m4
===================================================================
--- vegastrike-0.5.1~r1.orig/m4scripts/VS_CHECK_PYTHON.m4	2011-11-13 02:24:49.000000000 +0100
+++ vegastrike-0.5.1~r1/m4scripts/VS_CHECK_PYTHON.m4	2014-04-06 21:39:59.377424951 +0200
@@ -91,7 +91,7 @@
 if test "x${FOUND_PYTHON}" = "xyes";
 then
     FOUND_LIBPYTHON_SO=no
-    PYTHON_libchk="${PYTHON_LIBPATH} /usr/lib /usr/local/lib /usr/lib64 /usr/local/lib64 /usr/lib64/python${PYTHON_SHORT}/config /usr/local/lib64/python${PYTHON_SHORT}/config /sw/lib/python${PYTHON_SHORT}/config /usr/lib/python${PYTHON_SHORT} /usr/lib/python${PYTHON_SHORT}/config /usr/local/lib/python${PYTHON_SHORT} /usr/local/lib/python${PYTHON_SHORT}/config /lib/python2.2/config"
+    PYTHON_libchk="${PYTHON_LIBPATH} /usr/lib /usr/lib/x86_64-linux-gnu /usr/lib/i386-linux-gnu /usr/local/lib /usr/lib64 /usr/local/lib64 /usr/lib64/python${PYTHON_SHORT}/config /usr/local/lib64/python${PYTHON_SHORT}/config /sw/lib/python${PYTHON_SHORT}/config /usr/lib/python${PYTHON_SHORT} /usr/lib/python${PYTHON_SHORT}/config /usr/local/lib/python${PYTHON_SHORT} /usr/local/lib/python${PYTHON_SHORT}/config /lib/python2.2/config"
     for i in ${PYTHON_libchk};
     do
         if test "x$is_macosx" = "xyes" ; then        	
