Another GCC 4.7 compatibility patch
Index: vegastrike-0.5.1~r1/src/cmd/unit_jump.h
===================================================================
--- vegastrike-0.5.1~r1.orig/src/cmd/unit_jump.h	2012-10-20 16:29:04.043470000 +0200
+++ vegastrike-0.5.1~r1/src/cmd/unit_jump.h	2012-10-20 21:43:44.027062203 +0200
@@ -108,7 +108,7 @@
                      (tester = *i) != NULL; ++i)
                     if (tester->isUnit() == UNITPTR && tester != this)
                         if ( ( this->LocalPosition()-tester->LocalPosition() ).Magnitude() < this->rSize()+tester->rSize() )
-                            SetCurPosition( this->LocalPosition()+this->cumulative_transformation_matrix.getR()
+                            this->SetCurPosition( this->LocalPosition()+this->cumulative_transformation_matrix.getR()
                                            *( 4*( this->rSize()+tester->rSize() ) ) );
             DealPossibleJumpDamage( this );
             static int jumparrive = AUDCreateSound( vs_config->getVariable( "unitaudio", "jumparrive", "sfx43.wav" ), false );
