# add line from data package
Index: megaglest-3.11.0/mk/linux/glest.ini
===================================================================
--- megaglest-3.11.0.orig/mk/linux/glest.ini	2015-01-21 23:09:03.000000000 +0100
+++ megaglest-3.11.0/mk/linux/glest.ini	2015-01-24 22:53:38.408313939 +0100
@@ -93,3 +93,4 @@
 UserData_Root=$HOME/.megaglest/
 VersionURL=http://master.megaglest.org/files/versions/
 Windowed=false
+CountryTexturePath=/usr/share/flags/countries/16x11/
