# fix: https://bugs.launchpad.net/ubuntu/+source/libsdl2/+bug/1306629
Index: dhewm3-1.4.0/neo/CMakeLists.txt
===================================================================
--- dhewm3-1.4.0.orig/neo/CMakeLists.txt	2015-10-11 02:57:57.188194411 +0200
+++ dhewm3-1.4.0/neo/CMakeLists.txt	2015-10-11 02:57:57.184194255 +0200
@@ -115,6 +115,8 @@
 	endif()
 	find_package(SDL2 REQUIRED)
 	include_directories(${SDL2_INCLUDE_DIR})
+	include_directories(/usr/include/mirclient)
+	include_directories(/usr/include/mircommon)
 	set(SDLx_LIBRARY ${SDL2_LIBRARY})
 else()
 	# skip SDLmain
