# fix libbullet 2.81 compatibility
# add size_t namespace.
Index: racer-fzero-0.0.320/libtrack/document/MoveNodeDelta.h
===================================================================
--- racer-fzero-0.0.320.orig/libtrack/document/MoveNodeDelta.h	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/libtrack/document/MoveNodeDelta.h	2014-04-06 19:46:19.645483175 +0200
@@ -15,6 +15,7 @@
 #include "ChangePropertyDelta.h"
 
 #include <LinearMath/btVector3.h>
+#include <cstddef>
 
 namespace Document
 {
Index: racer-fzero-0.0.320/libtrack/document/MoveTrackAttachmentDelta.h
===================================================================
--- racer-fzero-0.0.320.orig/libtrack/document/MoveTrackAttachmentDelta.h	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/libtrack/document/MoveTrackAttachmentDelta.h	2014-04-06 19:46:19.645483175 +0200
@@ -15,6 +15,7 @@
 #include "ChangePropertyDelta.h"
 
 #include <LinearMath/btVector3.h>
+#include <cstddef>
 
 namespace Document
 {
Index: racer-fzero-0.0.320/racer/Engine/GameObjects/Car.cpp
===================================================================
--- racer-fzero-0.0.320.orig/racer/Engine/GameObjects/Car.cpp	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/racer/Engine/GameObjects/Car.cpp	2014-04-06 19:46:19.649483175 +0200
@@ -37,23 +37,24 @@
  */
 
 void contact_added_callback_obj (btManifoldPoint& cp,
-                                 const btCollisionObject* colObj,
+                                 const btCollisionObjectWrapper* col,
                                  int partId, int index)
 {
         (void) partId;
         (void) index;
-        const btCollisionShape *shape = colObj->getCollisionShape();
+	const btCollisionObject* obj = col->getCollisionObject();
+        const btCollisionShape *shape = col->getCollisionShape();
 
         if (shape->getShapeType() != TRIANGLE_SHAPE_PROXYTYPE) return;
         const btTriangleShape *tshape =
-               static_cast<const btTriangleShape*>(colObj->getCollisionShape());
+               static_cast<const btTriangleShape*>(col->getCollisionShape());
 
 
-        const btCollisionShape *parent = colObj->getRootCollisionShape();
+        const btCollisionShape *parent = obj->getCollisionShape();
         if (parent == NULL) return;
         if (parent->getShapeType() != TRIANGLE_MESH_SHAPE_PROXYTYPE) return;
 
-        btTransform orient = colObj->getWorldTransform();
+        btTransform orient = col->getWorldTransform();
         orient.setOrigin( btVector3(0.0f,0.0f,0.0f ) );
 
         btVector3 v1 = tshape->m_vertices1[0];
@@ -73,13 +74,13 @@
 }
 
 bool contact_added_callback (btManifoldPoint& cp,
-                             const btCollisionObject* colObj0,
+                             const btCollisionObjectWrapper* col0,
                              int partId0, int index0,
-                             const btCollisionObject* colObj1,
+                             const btCollisionObjectWrapper* col1,
                              int partId1, int index1)
 {
-        contact_added_callback_obj(cp, colObj0, partId0, index0);
-        contact_added_callback_obj(cp, colObj1, partId1, index1);
+        contact_added_callback_obj(cp, col0, partId0, index0);
+        contact_added_callback_obj(cp, col1, partId1, index1);
         //std::cout << to_ogre(cp.m_normalWorldOnB) << std::endl;
         return true;
 }
Index: racer-fzero-0.0.320/racer/Engine/GameObjects/Car.h
===================================================================
--- racer-fzero-0.0.320.orig/racer/Engine/GameObjects/Car.h	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/racer/Engine/GameObjects/Car.h	2014-04-06 19:46:19.649483175 +0200
@@ -24,6 +24,7 @@
 
 namespace Engine
 {
+    struct btCollisionObjectWrapper;
     class InputDevice;
 
 /// Objects which affect the gameplay.
Index: racer-fzero-0.0.320/configure.ac
===================================================================
--- racer-fzero-0.0.320.orig/configure.ac	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/configure.ac	2014-04-06 19:46:45.245482956 +0200
@@ -32,7 +32,7 @@
 fi
 
 #boost libraries build type
-AC_SUBST(BOOST_LIB_SUFFIX, ["-mt"])
+AC_SUBST(BOOST_LIB_SUFFIX, [""])
 AC_ARG_VAR(BOOST_LIB_SUFFIX, "Extension for boost library builds, for example -mt.")
 
 AC_SUBST(EXTRA_CFLAGS, ["$debug_specific_CFLAGS -Wall"])
@@ -52,7 +52,7 @@
 AC_SUBST(racer_partial_LIBS)
 AC_SUBST(racer_partial_CFLAGS)
 AC_SUBST(racer_CFLAGS, ["$EXTRA_CFLAGS $SDL_CFLAGS $racer_partial_CFLAGS"])
-AC_SUBST(racer_LIBS, ["$SDL_LIBS $racer_partial_LIBS -lSDL_image -lGLEW"])
+AC_SUBST(racer_LIBS, ["$SDL_LIBS $racer_partial_LIBS -lSDL_image -lGLEW -lGLU"])
 #old versions of FTGL use different headers.
 PKG_CHECK_EXISTS(ftgl >= 2.1.3,
   [AC_DEFINE(HAVE_FTGL_2_1_3, [1], [Defined if ftgl is at least version 2.1.3])],
Index: racer-fzero-0.0.320/configure
===================================================================
--- racer-fzero-0.0.320.orig/configure	2014-04-06 19:46:19.657483175 +0200
+++ racer-fzero-0.0.320/configure	2014-04-06 19:46:19.653483175 +0200
@@ -14340,7 +14340,7 @@
 fi
 
 #boost libraries build type
-BOOST_LIB_SUFFIX="-mt"
+BOOST_LIB_SUFFIX=""
 
 
 
