# fix build errors
diff --git a/Makefile b/Makefile
index dbbfe0b..0197dbd 100644
--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@ $(if $(shell test "$(GL)" -ge 3 -a "$(SDL)" -lt 2 && echo fail), \
      $(error "GL >= 3 only supported with SDL >= 2"))
 
 # library headers
-CFLAGS+= `pkg-config --cflags $(SDL_) $(LUA) $(LUA)-socket libenet libpng zlib openal`
+CFLAGS+= `pkg-config --cflags $(SDL_) $(LUA) $(LUA)-socket $(LUA)-mime libenet libpng zlib openal`
 ifeq ($(MACOSX),1)
   CFLAGS += -DMACOSX
 endif
@@ -95,7 +95,7 @@ ifeq ($(STATIC),1)
   LIB+= -Wl,-dn
   PKG_CFG_OPTS= --static
 endif
-LIB+= `pkg-config $(PKG_CFG_OPTS) --libs $(LUA) $(LUA)-socket libenet libpng zlib openal` -lm
+LIB+= `pkg-config $(PKG_CFG_OPTS) --libs $(LUA) $(LUA)-socket $(LUA)-mime libenet libpng zlib openal` -lm
 ifeq ($(STATIC),1)
   LIB+= -Wl,-dy
 endif
diff --git a/net_enet_2.c b/net_enet_2.c
index fbadb0a..b61ddc2 100644
--- a/net_enet_2.c
+++ b/net_enet_2.c
@@ -237,7 +237,7 @@ static network_return_t enet_setup( char* str_address, int port )
 	DebugPrintf("network: enet setup address %s\n",
 		address_to_str(&address));
 
-	enet_host = enet_host_create( &address, max_peers, 0, 0 );
+	enet_host = enet_host_create( &address, max_peers, 0, 0, 0 );
 
 	if ( enet_host == NULL )
 	{
@@ -267,7 +267,7 @@ static int enet_connect( char* str_address, int port )
 	DebugPrintf("network: enet connect to address %s\n",
 		address_to_str(&address));
 
-	peer = enet_host_connect( enet_host, &address, max_channels );
+	peer = enet_host_connect( enet_host, &address, max_channels, 0 );
 
 	if (peer == NULL)
 	{
@@ -1311,7 +1311,7 @@ static void new_packet( ENetEvent * event )
 					network_peer_data_t * new_peer_data;
 					DebugPrintf("network: host told us to connect to player %d address %s\n",
 						packet->id, address_to_str( address ));
-					new_peer = enet_host_connect( enet_host, address, max_channels );
+					new_peer = enet_host_connect( enet_host, address, max_channels, 0 );
 					if(!new_peer)
 					{
 						DebugPrintf("network: enet host connect returned NULL.\n");
