# wrong path
Index: scummvm-1.7.0/configure
===================================================================
--- scummvm-1.7.0.orig/configure	2014-07-24 20:52:21.247986483 +0200
+++ scummvm-1.7.0/configure	2014-07-26 00:12:09.296954635 +0200
@@ -3845,8 +3845,8 @@
 			;;
 		*)
 			# Unity has a lots of dependencies, update the libs and cflags var with them
-			LIBUNITY_LIBS="$LIBUNITY_LIBS `pkg-config --libs unity = 3.8.4 2>> "$TMPLOG"`"
-			LIBUNITY_CFLAGS="$LIBUNITY_CFLAGS `pkg-config --cflags unity = 3.8.4 2>> "$TMPLOG"`"
+			LIBUNITY_LIBS="$LIBUNITY_LIBS `pkg-config --libs unity 2>> "$TMPLOG"`"
+			LIBUNITY_CFLAGS="$LIBUNITY_CFLAGS `pkg-config --cflags unity 2>> "$TMPLOG"`"
 			_libunity=no
 			cat > $TMPC << EOF
 #include <unity.h>
