# add geoip
Index: openra-20130915/Makefile
===================================================================
--- openra-20130915.orig/Makefile	2013-09-14 13:28:28.000000000 +0200
+++ openra-20130915/Makefile	2013-09-15 15:00:30.627786107 +0200
@@ -54,7 +54,7 @@
 INSTALL_DATA = $(INSTALL) -m644
 
 # program targets
-CORE = fileformats rcg rgl rsdl rnull game utility
+CORE = fileformats rcg rgl rsdl rnull game utility geoip
 TOOLS = editor tsbuild ralint
 
 VERSION     = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
