# install game files into /usr/share/games/openra
Index: openra-20150919/Makefile
===================================================================
--- openra-20150919.orig/Makefile	2015-09-19 15:29:13.230708195 +0200
+++ openra-20150919/Makefile	2015-09-19 15:29:13.226709738 +0200
@@ -61,7 +61,7 @@
 mandir ?= $(datarootdir)/man/
 bindir ?= $(prefix)/bin
 libdir ?= $(prefix)/lib
-gameinstalldir ?= $(libdir)/openra
+gameinstalldir ?= $(datadir)/openra
 
 BIN_INSTALL_DIR = $(DESTDIR)$(bindir)
 DATA_INSTALL_DIR = $(DESTDIR)$(gameinstalldir)
