# install game files into /usr/share/games/openra
Index: openra-20150424/Makefile
===================================================================
--- openra-20150424.orig/Makefile
+++ openra-20150424/Makefile
@@ -52,7 +52,7 @@ datarootdir ?= $(prefix)/share
 datadir ?= $(datarootdir)
 bindir ?= $(prefix)/bin
 libdir ?= $(prefix)/lib
-gameinstalldir ?= $(libdir)/openra
+gameinstalldir ?= $(datadir)/openra
 
 BIN_INSTALL_DIR = $(DESTDIR)$(bindir)
 DATA_INSTALL_DIR = $(DESTDIR)$(gameinstalldir)
