Just some typos.
Index: hedgewars-0.9.18/QTfrontend/res/css/chat.css
===================================================================
--- hedgewars-0.9.18.orig/QTfrontend/res/css/chat.css	2012-07-31 21:29:31.000000000 +0200
+++ hedgewars-0.9.18/QTfrontend/res/css/chat.css	2012-11-01 19:52:43.255138477 +0100
@@ -9,7 +9,7 @@
  *
  * In the QTfrontend of hedgewars also display:none; will work for class names
  * that start with msg_ and .timestamp -  as long as they are referenced
- * directly and not within any class hierachy.
+ * directly and not within any class hierarchy.
  * Note: Will only effect new lines!
  *
  ******************************************************************************
Index: hedgewars-0.9.18/QTfrontend/ui/widget/chatwidget.cpp
===================================================================
--- hedgewars-0.9.18.orig/QTfrontend/ui/widget/chatwidget.cpp	2012-10-17 22:26:19.000000000 +0200
+++ hedgewars-0.9.18/QTfrontend/ui/widget/chatwidget.cpp	2012-11-01 19:53:12.935137591 +0100
@@ -109,7 +109,7 @@
     // matches all {...} and , (used as seperator for splitting into names)
     QRegExp split(" *(\\{[^}]*\\}|,) *");
 
-    // matches class names that are referenced without hierachy
+    // matches class names that are referenced without hierarchy
     QRegExp nohierarchy("^.[^ .]+$");
 
     QStringList victims = QString(style).
