# format is mandatory
Index: naev-0.6.0/src/dev_system.c
===================================================================
--- naev-0.6.0.orig/src/dev_system.c	2015-02-16 18:49:03.000000000 +0100
+++ naev-0.6.0/src/dev_system.c	2015-03-17 19:24:10.639624581 +0100
@@ -247,7 +247,7 @@
    for (i = 0; i < uniedit_nsys; i++) {
       s = uniedit_sys[i];
       xmlw_startElem( writer, "sys" );
-      xmlw_attr( writer, "name", s->name );
+      xmlw_attr( writer, "name", "%s", s->name );
 
       /* Iterate jumps and see if they lead to any other systems in our array. */
       for (j = 0; j < s->njumps; j++) {
Index: naev-0.6.0/src/log.c
===================================================================
--- naev-0.6.0.orig/src/log.c	2015-03-05 23:11:11.000000000 +0100
+++ naev-0.6.0/src/log.c	2015-03-17 19:26:25.659620564 +0100
@@ -191,10 +191,10 @@
    }
 
    if (noutcopy)
-      fprintf( stdout, outcopy );
+      fprintf( stdout, "%s", outcopy );
 
    if (nerrcopy)
-      fprintf( stderr, errcopy );
+      fprintf( stderr, "%s", errcopy );
 
    log_purge();
 }
