# apply boost fixes
Index: violetland-0.5/src/game/Highscores.h
===================================================================
--- violetland-0.5.orig/src/game/Highscores.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/game/Highscores.h	2016-01-04 21:42:17.872679779 +0100
@@ -3,6 +3,13 @@
 
 #include <set>
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
 
Index: violetland-0.5/src/game/WeaponManager.h
===================================================================
--- violetland-0.5.orig/src/game/WeaponManager.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/game/WeaponManager.h	2016-01-04 21:42:17.872679779 +0100
@@ -1,6 +1,13 @@
 #ifndef WEAPONMANAGER_H_
 #define WEAPONMANAGER_H_
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem/fstream.hpp>
 
 #include "../system/sound/SoundManager.h"
Index: violetland-0.5/src/game/lifeforms/MonsterFactory.h
===================================================================
--- violetland-0.5.orig/src/game/lifeforms/MonsterFactory.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/game/lifeforms/MonsterFactory.h	2016-01-04 21:42:17.872679779 +0100
@@ -3,6 +3,13 @@
 
 #include <sstream>
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem/fstream.hpp>
 
 #include "../../system/sound/SoundManager.h"
Index: violetland-0.5/src/program.cpp
===================================================================
--- violetland-0.5.orig/src/program.cpp	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/program.cpp	2016-01-04 21:42:17.872679779 +0100
@@ -29,6 +29,13 @@
 #include "SDL_ttf.h"
 
 // Boost
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 #include <boost/format.hpp>
 
Index: violetland-0.5/src/system/ConfigFile.h
===================================================================
--- violetland-0.5.orig/src/system/ConfigFile.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/ConfigFile.h	2016-01-04 21:42:17.872679779 +0100
@@ -7,6 +7,13 @@
 #include <iostream>
 #include <sstream>
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
 
Index: violetland-0.5/src/system/Configuration.h
===================================================================
--- violetland-0.5.orig/src/system/Configuration.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/Configuration.h	2016-01-04 21:42:17.872679779 +0100
@@ -1,6 +1,13 @@
 #ifndef CONFIGURATION_H_
 #define CONFIGURATION_H_
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem/fstream.hpp>
 
 #include "utility/FileUtility.h"
Index: violetland-0.5/src/system/graphic/text/TextManager.h
===================================================================
--- violetland-0.5.orig/src/system/graphic/text/TextManager.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/graphic/text/TextManager.h	2016-01-04 21:42:17.872679779 +0100
@@ -1,6 +1,13 @@
 #ifndef TEXTMANAGER_H_
 #define TEXTMANAGER_H_
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 
 #include "TextObject.h"
Index: violetland-0.5/src/system/sound/SoundManager.h
===================================================================
--- violetland-0.5.orig/src/system/sound/SoundManager.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/sound/SoundManager.h	2016-01-04 21:42:17.872679779 +0100
@@ -1,6 +1,13 @@
 #ifndef SOUNDMANAGER_H_
 #define SOUNDMANAGER_H_
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 
 #include "Sound.h"
Index: violetland-0.5/src/system/utility/FileUtility.h
===================================================================
--- violetland-0.5.orig/src/system/utility/FileUtility.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/utility/FileUtility.h	2016-01-04 21:42:17.872679779 +0100
@@ -6,6 +6,13 @@
 #include <cstdlib>
 #include <string>
 
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 
 
Index: violetland-0.5/src/system/utility/ImageUtility.h
===================================================================
--- violetland-0.5.orig/src/system/utility/ImageUtility.h	2016-01-04 21:42:17.876679881 +0100
+++ violetland-0.5/src/system/utility/ImageUtility.h	2016-01-04 21:42:17.872679779 +0100
@@ -2,6 +2,13 @@
 #define IMAGEUTILITY_H_
 
 #include <iostream>
+// as of boost 1.50 there is no version 2, and compiles will fail 
+#include <boost/version.hpp> 
+#if BOOST_VERSION > 104900 
+#define BOOST_FILESYSTEM_VERSION 3 
+#else 
+#define BOOST_FILESYSTEM_VERSION 2 
+#endif
 #include <boost/filesystem.hpp>
 #include "SDL_ttf.h"
 #include "SDL_image.h"
