# get the version from env var. not from git.
Index: springlobby-0.194/tools/regen_config_header.sh
===================================================================
--- springlobby-0.194.orig/tools/regen_config_header.sh	2014-05-23 03:45:33.340002194 +0200
+++ springlobby-0.194/tools/regen_config_header.sh	2014-05-23 03:45:33.336002194 +0200
@@ -11,11 +11,7 @@
 SOURCE="${1}"
 OUTPUT="${2}"
 
-REV="$(git describe --tags)" 2>/dev/null
-
-if [ -z "$REV" ]; then
-	REV="unknown"
-fi
+REV="${SPRINGLOBBY_REV}"
 
 OLDREV=""
 if [ -s ${SOURCE}/VERSION ] && [ -s ${SOURCE}/springlobby_config.h ]; then
