# get the version from env var. not from git.
Index: springlobby-0.193/tools/regen_config_header.sh
===================================================================
--- springlobby-0.193.orig/tools/regen_config_header.sh	2014-05-13 10:56:28.000000000 +0200
+++ springlobby-0.193/tools/regen_config_header.sh	2014-05-22 19:10:52.000000000 +0200
@@ -14,11 +14,7 @@
 
 REV_TEMPLATE="@SPRINGLOBBY_REV@"
 
-REV="$(git describe --tags)" 2>/dev/null
-
-if [ -z "$REV" ]; then
-	REV="unknown"
-fi
+REV="${SPRINGLOBBY_REV}"
 
 OLDREV=""
 if [ -s ${VERSIONFILE} ] && [ -s ${TARGET_HEADER} ]; then
