Do not run tasks handled by the package.
Index: snowstorm-3.5.2.273873/secondlife
===================================================================
--- snowstorm-3.5.2.273873.orig/secondlife	2013-04-12 18:52:30.000000000 +0200
+++ snowstorm-3.5.2.273873/secondlife	2013-04-13 13:05:07.675677219 +0200
@@ -92,10 +92,10 @@
 cd "${RUN_PATH}"
 
 # Re-register the secondlife:// protocol handler every launch, for now.
-./etc/register_secondlifeprotocol.sh
+#./etc/register_secondlifeprotocol.sh
 
 # Re-register the application with the desktop system every launch, for now.
-./etc/refresh_desktop_app_entry.sh
+#./etc/refresh_desktop_app_entry.sh
 
 ## Before we mess with LD_LIBRARY_PATH, save the old one to restore for
 ##  subprocesses that care.
