Do not run tasks handled by the package.
Index: snowstorm-3.4.2.265141/secondlife
===================================================================
--- snowstorm-3.4.2.265141.orig/secondlife	2012-09-25 20:31:21.000000000 +0200
+++ snowstorm-3.4.2.265141/secondlife	2012-09-25 22:22:11.482771381 +0200
@@ -83,10 +83,10 @@
 cd "${RUN_PATH}"
 
 # Re-register the secondlife:// protocol handler every launch, for now.
-./etc/register_secondlifeprotocol.sh
+#./etc/register_secondlifeprotocol.sh
 
 # Re-register the application with the desktop system every launch, for now.
-./etc/refresh_desktop_app_entry.sh
+#./etc/refresh_desktop_app_entry.sh
 
 ## Before we mess with LD_LIBRARY_PATH, save the old one to restore for
 ##  subprocesses that care.
