Just a typo.
Index: xonotic-0.8.2/source/darkplaces/cmd.c
===================================================================
--- xonotic-0.8.2.orig/source/darkplaces/cmd.c	2017-04-03 18:12:51.284322154 +0200
+++ xonotic-0.8.2/source/darkplaces/cmd.c	2017-04-03 18:12:51.280322233 +0200
@@ -1614,7 +1614,7 @@
 			l = (int)strlen(com_token) + 1;
 			if (cmd_tokenizebufferpos + l > CMD_TOKENIZELENGTH)
 			{
-				Con_Printf("Cmd_TokenizeString: ran out of %i character buffer space for command arguements\n", CMD_TOKENIZELENGTH);
+				Con_Printf("Cmd_TokenizeString: ran out of %i character buffer space for command arguments\n", CMD_TOKENIZELENGTH);
 				break;
 			}
 			memcpy (cmd_tokenizebuffer + cmd_tokenizebufferpos, com_token, l);
