Just typos.
Index: xmoto-0.5.8/src/GameText.h
===================================================================
--- xmoto-0.5.8.orig/src/GameText.h	2011-09-28 23:56:41.107096425 +0200
+++ xmoto-0.5.8/src/GameText.h	2011-09-28 23:57:12.377096435 +0200
@@ -379,7 +379,7 @@
 #define GAMETEXT_MAIN                 _("Main")
 #define GAMETEXT_MOUSE                _("Mouse")
 #define GAMETEXT_THEME                _("Theme")
-#define GAMETEXT_TTF_MISSING          "You must install the font file for your langage:" // not translated because always displayed in english
+#define GAMETEXT_TTF_MISSING          "You must install the font file for your language:" // not translated because always displayed in english
 #define GAMETEXT_MEDAL_BRONZE         _("Bronze")
 #define GAMETEXT_MEDAL_GOLD           _("Gold")
 #define GAMETEXT_MEDAL_SILVER         _("Silver")
Index: xmoto-0.5.8/src/VTexture.cpp
===================================================================
--- xmoto-0.5.8.orig/src/VTexture.cpp	2011-09-28 23:55:13.577096395 +0200
+++ xmoto-0.5.8/src/VTexture.cpp	2011-09-28 23:56:07.177096414 +0200
@@ -377,7 +377,7 @@
       int i=1;
       while(it != textures.end()){
 	if((*it)->curRegistrationStageMode != RSM_NORMAL || (*it)->curRegistrationStage.size() > 0) {
-	  LogDebug("%3i   %s (%s)", i, (*it)->Name.c_str(), (*it)->curRegistrationStageMode == RSM_NORMAL ? "NORMAL" : "PERSISTANT");
+	  LogDebug("%3i   %s (%s)", i, (*it)->Name.c_str(), (*it)->curRegistrationStageMode == RSM_NORMAL ? "NORMAL" : "PERSISTENT");
 	  i++;
 	}
 	++it;
Index: xmoto-0.5.8/src/db/xmDatabase_sync.cpp
===================================================================
--- xmoto-0.5.8.orig/src/db/xmDatabase_sync.cpp	2011-09-28 23:58:00.827096451 +0200
+++ xmoto-0.5.8/src/db/xmDatabase_sync.cpp	2011-09-28 23:59:31.007096482 +0200
@@ -57,7 +57,7 @@
 		      "WHERE sitekey=\"" + protectString(i_sitekey) + "\" AND id_profile=\"" + protectString(i_profile) + "\";", nrow);
     if(nrow != 1) {
       read_DB_free(v_result);
-      throw Exception("Unable to retrieve informations");
+      throw Exception("Unable to retrieve information");
     }
 
     snprintf(v_line, 2048,
Index: xmoto-0.5.8/xmoto.6
===================================================================
--- xmoto-0.5.8.orig/xmoto.6	2011-09-28 23:50:25.047096299 +0200
+++ xmoto-0.5.8/xmoto.6	2011-09-28 23:52:48.237096347 +0200
@@ -84,7 +84,7 @@
 Switch to the "Ugly mode". Saves a lot of CPU time and shows some secret passages. 
 .TP
 .B F10 
-Theme Mode. It can be used to test custom graphics. Also helps to avoid wreckers.
+Theme Mode. It can be used to test custom graphics. Also helps one to avoid wreckers.
 .TP 
 .B F11 
 Line Mode. This mode is mostly for testing. Draws lines around blocks and shows zones.
@@ -183,7 +183,7 @@
 Tries to render the game as fast as possible. No delaying, maximum framerate.
 .TP
 .B --benchmark
-Only meaningful when combined with --replay and --timedemo. Useful to determine graphics performance.
+Only meaningful when combined with \-\-replay and \-\-timedemo. Useful to determine graphics performance.
 .TP
 .B --nowww
 Do not  allow xmoto to connect to the web.
@@ -209,16 +209,16 @@
 as default theme for new profiles created. This is mostly useful for packagers wanting to set the theme of their distribution by default.
 .TP
 .B --noDBDirsCheck
-Don't check that system and user dirs changed at startup. It is mainly usefull if you use X-Moto on a USB key.
+Don't check that system and user dirs changed at startup. It is mainly useful if you use X-Moto on a USB key.
 .TP
 .B --server
 Run X-Moto as a server only (no gui).
 .TP
 .B --serverPort <port>
-Specify the server port (with --server only).
+Specify the server port (with \-\-server only).
 .TP
 .B --serverAdminPassword <password>
-Specify a server admin password which is always valid (with --server only).
+Specify a server admin password which is always valid (with \-\-server only).
 .TP
 .B --connectAtStartup
 Connect the client to the server at startup.
@@ -257,7 +257,7 @@
 Enable sql trace mode.
 .TP
 .B      --videoRecording
-Enable video recording while plaing/replaying (should be used with --replay and --benchmark).
+Enable video recording while plaing/replaying (should be used with \-\-replay and \-\-benchmark).
 .TP
 .B       --videoRecordingSizeDivision DIVISION
 Change video size (1=full, 2=50%, 4=25%).
@@ -272,16 +272,16 @@
 Stop recording video after NBCENTSOFSECONDS time elapsed.
 .TP
 .B	--hidePlayingInformation
-Don't show some information while playing/replaying ; usefull to make nicer video.
+Don't show some information while playing/replaying ; useful to make nicer video.
 
 .SH EXAMPLES
 .B view a replay in windowed mode:
 .br
-xmoto --windowed ~/.xmoto/Replays/replay.rpl
+xmoto \-\-windowed ~/.xmoto/Replays/replay.rpl
 .br
 .B disable sounds, Internet connection and play mylevel.lvl directly:
 .br
-xmoto --nosound --nowww mylevel.lvl
+xmoto \-\-nosound \-\-nowww mylevel.lvl
 
 .\".SH SEE ALSO
 .\" xmoto-edit is deprecated. Should I leave it here and write "deprecated, use inksmoto"?
Index: xmoto-0.5.8/src/XMArgs.cpp
===================================================================
--- xmoto-0.5.8.orig/src/XMArgs.cpp	2011-09-29 00:05:42.147096607 +0200
+++ xmoto-0.5.8/src/XMArgs.cpp	2011-09-29 00:05:52.517096610 +0200
@@ -691,7 +691,7 @@
   printf("\t--videoRecordingFramerate FRAMERATE\n\t\tChange video framerate.\n");
   printf("\t--videoRecordingStartTime NBCENTSOFSECONDS\n\t\tStart recording video after this game time.\n");
   printf("\t--videoRecordingEndTime NBCENTSOFSECONDS\n\t\tStop recording video after this game time.\n");
-  printf("\t--hidePlayingInformation\n\t\tDon't show some information while playing/replaying ; usefull to make nicer video.\n");
+  printf("\t--hidePlayingInformation\n\t\tDon't show some information while playing/replaying ; useful to make nicer video.\n");
   printf("\t--drawlib DRAWLIB\n\t\tChoose the render to use (default one is OPENGL if available).\n");
   printf("\t--defaultTheme THEME\n\t\tDefault theme for new profiles created.\n");
   printf("\t--noDBDirsCheck\n\t\tDon't check that system and user dirs changed at startup.\n");
