From 9876a3f7f48f6083f678e86ea1eca8f5fc13e0e3 Mon Sep 17 00:00:00 2001
From: Pascal de Bruijn <pmjdebruijn@pcode.nl>
Date: Fri, 28 Mar 2014 18:40:24 +0100
Subject: [PATCH] menus_settings: add anisotropic filtering

---
 config/menus_settings.cfg | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/config/menus_settings.cfg b/config/menus_settings.cfg
index 59aae52..80b79f4 100644
--- a/config/menus_settings.cfg
+++ b/config/menus_settings.cfg
@@ -110,6 +110,7 @@ menuitem "2			        [ OK ]" [ closemenu [QUICK setup]; resetgl ]
 const new_gfx_quality [
   if (= $arg1 6) [ // "INSANE!" gfx quality
     fsaa 8
+    aniso 8
     minlod 250
     gfxqualityalias = 6
     fpsrange 100 1000
@@ -134,6 +135,7 @@ const new_gfx_quality [
   ] [
     if (= $arg1 5) [ // High gfx quality
       fsaa 4
+      aniso 4
       minlod 200
       gfxqualityalias = 5
       fpsrange 100 1000
@@ -157,6 +159,7 @@ const new_gfx_quality [
     ] [
       if (= $arg1 4) [ // Good gfx quality
         fsaa 0
+        aniso 0
         minlod 100
         gfxqualityalias = 4
         fpsrange 60 60
@@ -180,6 +183,7 @@ const new_gfx_quality [
       ] [
         if (= $arg1 3) [ // Medium gfx quality (DEFAULT)
           fsaa 0
+          aniso 0
           minlod 60
           gfxqualityalias = 3
           fpsrange 30 40
@@ -208,6 +212,7 @@ const new_gfx_quality [
         ] [
           if (= $arg1 2) [ // Low gfx quality
             fsaa 0
+            aniso 0
             minlod 60
             gfxqualityalias = 2
             fpsrange 20 25
@@ -229,6 +234,7 @@ const new_gfx_quality [
             hidebigmenuimages 1
           ] [ // Horrible gfx quality
             fsaa 0
+            aniso 0
             minlod 60
             gfxqualityalias = 1
             fpsrange 20 25
@@ -354,6 +360,7 @@ menuitem [] -1
 menuitem "2			  [ Apply now! ]"   [ resetgl ]
 
 const fsaa_valuemap [-1 2 4 8 16]
+const aniso_valuemap [0 2 4 8 16]
 const minlod_valuemap [60 100 200 250]
 const watersubdiv_valuemap [16 8 4 2 1]
 
@@ -361,6 +368,7 @@ newmenu [Advanced video settings]
 
 menuitemslider [Scope FOV: ] 5 60 "$scopefov" 5 [] [ scopefov $arg1 ]
 menuitemslider [Antialiasing (FSAA): ] 0 4 "(findlist $fsaa_valuemap $fsaa)" 1 [Off 2 4 8 16] [ fsaa (at $fsaa_valuemap $arg1) ]
+menuitemslider [Anisotropic filtering: ] 0 4 "(findlist $aniso_valuemap $aniso)" 1 [Off 2 4 8 16] [ aniso (at $aniso_valuemap $arg1) ]
 menuitemslider [Minimal level of detail: ] 0 3 "(findlist $minlod_valuemap $minlod)" 1 [Low Decent Good Best] [ minlod (at $minlod_valuemap $arg1) ]
 menuitemslider [Light error: ] 1 100 "$lighterror" 5 [] [ lighterror $arg1 ]
 menuitemslider [Water subdivision: ] 0 4 "(findlist $watersubdiv_valuemap $watersubdiv)" 1 ["Lowest quality" "Low quality" "Medium quality" "High quality" "Highest quality"] [ watersubdiv (at $watersubdiv_valuemap $arg1) ]
-- 
1.9.1

