# change installation directories
Index: trigger-rally-0.6.3/src/GNUmakefile
===================================================================
--- trigger-rally-0.6.3.orig/src/GNUmakefile	2016-01-29 22:06:52.000000000 +0100
+++ trigger-rally-0.6.3/src/GNUmakefile	2016-03-22 22:17:32.902940619 +0100
@@ -9,9 +9,9 @@
 DISTARC         := $(DISTDIR).tar.gz
 TR_EXENAME      := trigger-rally
 TR_CFGNAME      := trigger-rally.config.defs
-TR_BINDIR       := ../bin
-TR_DATADIR      := ../data
-TR_DOCDIR       := ../doc
+TR_BINDIR       := .
+TR_DATADIR      := .
+TR_DOCDIR       := /usr/share/doc/trigger-rally
 TR_EXEFILE      := $(TR_BINDIR)/$(TR_EXENAME)
 TR_CFGFILE      := $(TR_BINDIR)/$(TR_CFGNAME)
 TR_DATAFILES    := $(shell cd $(TR_DATADIR); find * -type f)
