Fix broken SVN detection.
Index: simgear-2.10.0/CMakeModules/FindSvnClient.cmake
===================================================================
--- simgear-2.10.0.orig/CMakeModules/FindSvnClient.cmake	2013-01-28 18:02:28.000000000 +0100
+++ simgear-2.10.0/CMakeModules/FindSvnClient.cmake	2013-02-19 15:06:25.198139997 +0100
@@ -55,7 +55,6 @@
 
 if(HAVE_APR_CONFIG OR MSVC)
 	find_path(LIBSVN_INCLUDE_DIR svn_client.h
-      NO_DEFAULT_PATH
 	  HINTS
 	  $ENV{LIBSVN_DIR} ${CMAKE_INSTALL_PREFIX} ${MSVC_3RDPARTY_ROOT}/${MSVC_3RDPARTY_DIR}/include
 	  PATH_SUFFIXES include/subversion-1
