Fix some defintiion and an include
Index: simgear-2.10.0/simgear/debug/logstream.hxx
===================================================================
--- simgear-2.10.0.orig/simgear/debug/logstream.hxx	2013-02-18 15:58:14.000000000 +0100
+++ simgear-2.10.0/simgear/debug/logstream.hxx	2013-02-19 15:04:49.610142840 +0100
@@ -31,6 +31,7 @@
 #  include <windows.h>
 #endif
 
+#include <cstdio>
 #include <streambuf>
 #include <ostream>
 #include <cstdio>
