# ft2build.h is not found without it
Index: freeminer-0.4.13.7/src/CMakeLists.txt
===================================================================
--- freeminer-0.4.13.7.orig/src/CMakeLists.txt	2015-10-21 21:47:16.000000000 +0200
+++ freeminer-0.4.13.7/src/CMakeLists.txt	2015-10-22 20:37:26.069098648 +0200
@@ -328,25 +328,25 @@
 ## CMake. If issues persist, re-enable this workaround specificially for the
 ## failing platforms.
 ##
-#	if(${CMAKE_SYSTEM} MATCHES "FreeBSD")
-#		#PkgConfig not worked. result: /usr/bin/ld: cannot find -lfreetype
-#	elseif(UNIX)
-#		include(FindPkgConfig)
-#		if(PKG_CONFIG_FOUND)
-#			pkg_check_modules(FREETYPE QUIET freetype2)
-#			if(FREETYPE_FOUND)
-#				SET(FREETYPE_PKGCONFIG_FOUND TRUE)
-#				SET(FREETYPE_LIBRARY ${FREETYPE_LIBRARIES})
-#				# Because CMake is idiotic
-#				string(REPLACE ";" " " FREETYPE_CFLAGS_STR ${FREETYPE_CFLAGS})
-#				string(REPLACE ";" " " FREETYPE_LDFLAGS_STR ${FREETYPE_LDFLAGS})
-#			endif(FREETYPE_FOUND)
-#		endif(PKG_CONFIG_FOUND)
-#	endif()
-#	if(NOT FREETYPE_FOUND)
-#		find_package(Freetype)
-#	endif()
-	find_package(Freetype)
+	if(${CMAKE_SYSTEM} MATCHES "FreeBSD")
+		#PkgConfig not worked. result: /usr/bin/ld: cannot find -lfreetype
+	elseif(UNIX)
+		include(FindPkgConfig)
+		if(PKG_CONFIG_FOUND)
+			pkg_check_modules(FREETYPE QUIET freetype2)
+			if(FREETYPE_FOUND)
+				SET(FREETYPE_PKGCONFIG_FOUND TRUE)
+				SET(FREETYPE_LIBRARY ${FREETYPE_LIBRARIES})
+				# Because CMake is idiotic
+				string(REPLACE ";" " " FREETYPE_CFLAGS_STR ${FREETYPE_CFLAGS})
+				string(REPLACE ";" " " FREETYPE_LDFLAGS_STR ${FREETYPE_LDFLAGS})
+			endif(FREETYPE_FOUND)
+		endif(PKG_CONFIG_FOUND)
+	endif()
+	if(NOT FREETYPE_FOUND)
+		find_package(Freetype)
+	endif()
+#	find_package(Freetype)
 
 	if(FREETYPE_FOUND)
 		message(STATUS "Freetype enabled.")
