Description: Use font from ttf-dejavu-core
Forwarded: not-needed
Author: Sven Eckelmann <sven@narfation.org>

---
diff --git a/projects/unix/Makefile b/projects/unix/Makefile
index e2df2c65e2dbceb0d7f7bc3854056e98c867132a..919355785628c10cdcf6ffc6787f882279db69af 100644
--- a/projects/unix/Makefile
+++ b/projects/unix/Makefile
@@ -423,6 +423,7 @@ install: $(TARGET)
 	$(INSTALL) -m 0644 $(INSTALL_STRIP_FLAG) $(TARGET) "$(DESTDIR)$(LIBDIR)"
 	$(INSTALL) -d "$(DESTDIR)$(SHAREDIR)"
 	$(INSTALL) -m 0644 ../../data/* "$(DESTDIR)$(SHAREDIR)"
+	$(RM) "$(DESTDIR)$(SHAREDIR)"/font.ttf
 	$(INSTALL) -d "$(DESTDIR)$(INCDIR)"
 	$(INSTALL) -m 0644 ../../src/api/m64p_*.h "$(DESTDIR)$(INCDIR)"
 	-$(LDCONFIG) "$(DESTDIR)$(LIBDIR)"
diff --git a/src/osd/osd.cpp b/src/osd/osd.cpp
index 77348d8c2388403f7a41dba94952e85895670e87..e0853fd8845d48b2a92fbb0d7ac31b4f60be63f1 100644
--- a/src/osd/osd.cpp
+++ b/src/osd/osd.cpp
@@ -39,7 +39,7 @@ extern "C" {
     #include "plugin/plugin.h"
 }
 
-#define FONT_FILENAME "font.ttf"
+#define FONT_FILENAME "/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans.ttf"
 
 typedef void (APIENTRYP PTRGLACTIVETEXTURE)(GLenum texture);
 static PTRGLACTIVETEXTURE pglActiveTexture = NULL;
@@ -222,7 +222,7 @@ void osd_init(int width, int height)
         return;
     }
 
-    fontpath = ConfigGetSharedDataFilepath(FONT_FILENAME);
+    fontpath = FONT_FILENAME;
 
     l_font = new OGLFT::Monochrome(fontpath, (float) height / 35.0f);  // make font size proportional to screen height
 
