Description: Remove unused variable bdslot
Origin: upstream, https://bitbucket.org/richard42/mupen64plus-core/changeset/382c921de4d0
Author: Sven Eckelmann <sven@narfation.org>

---
diff --git a/src/debugger/dbg_decoder.c b/src/debugger/dbg_decoder.c
index d5bf75c25061fd34c4d08298e1746c6f05cb8690..ece9f7423076b27a93d7b20427d700354e847bce 100644
--- a/src/debugger/dbg_decoder.c
+++ b/src/debugger/dbg_decoder.c
@@ -288,7 +288,6 @@ db_disasm_insn ( struct r4k_dis_t * state,
                  bool altfmt               )
 {
     char * rel;
-    bool bdslot = false;
     InstFmt i;
 
     i.word = insn;
@@ -348,7 +347,6 @@ db_disasm_insn ( struct r4k_dis_t * state,
             case OP_JR:
             case OP_JALR:
                     db_printf(state, "%s", r4k_str_reg_name[i.RType.rs]);
-                    bdslot = true;
                     break;
             case OP_MTLO:
             case OP_MTHI:
@@ -432,7 +430,6 @@ db_disasm_insn ( struct r4k_dis_t * state,
                 r4k_str_reg_name[i.IType.rt]);
     pr_displ:
             print_addr( state, loc + 4 + ((short)i.IType.imm << 2) );
-            bdslot = true;
             break;
 
     case OP_COP0:
@@ -566,7 +563,6 @@ db_disasm_insn ( struct r4k_dis_t * state,
     case OP_JAL:
             db_printf(state, "%-16s", r4k_str_op_name[i.JType.op]);
             print_addr(state, (loc & 0xF0000000) | (i.JType.target << 2));
-            bdslot = true;
             break;
     
     case OP_LDC1:
